app/src/main/java/co/nimblehq/compose/crypto/ui/screens/detail/DetailScreen.kt

```diff
@@ -10,22 +10,49 @@ import androidx.compose.ui.layout.onGloballyPositioned
 import androidx.compose.ui.platform.LocalDensity
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
 import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.navigation.NavController
 import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.extension.toFormattedString
 import co.nimblehq.compose.crypto.ui.common.price.PriceChangeButton
+import co.nimblehq.compose.crypto.ui.preview.DetailScreenParams
+import co.nimblehq.compose.crypto.ui.preview.DetailScreenPreviewParameterProvider
 import co.nimblehq.compose.crypto.ui.theme.*
 import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp0
 import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp16
 import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp60
 import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp8
 import co.nimblehq.compose.crypto.ui.theme.Style
 import co.nimblehq.compose.crypto.ui.theme.Style.textColor
+import co.nimblehq.compose.crypto.ui.uimodel.CoinDetailUiModel
 import coil.compose.rememberAsyncImagePainter
 
 @Composable
-fun DetailScreen() {
+fun DetailScreen(
+    navController: NavController,
+    viewModel: DetailViewModel,
+    id: String,
+) {
+    viewModel.input.getCoinId(coinId = id)
```

Comment:
It does not seem to be the right way to execute the input in composable directly like this. It produces the infinity API call loop as the log below:

<details>
<summary>Logcat</summary>

```
2022-09-29 23:17:12.785 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:12.786 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:12.810 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:12.810 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:13.134 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- 200 https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false (348ms)
2022-09-29 23:17:13.134 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  date: Thu, 29 Sep 2022 16:17:12 GMT
2022-09-29 23:17:13.135 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  content-type: application/json; charset=utf-8
2022-09-29 23:17:13.135 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-frame-options: SAMEORIGIN
2022-09-29 23:17:13.135 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-xss-protection: 1; mode=block
2022-09-29 23:17:13.135 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-content-type-options: nosniff
2022-09-29 23:17:13.135 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-download-options: noopen
2022-09-29 23:17:13.135 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-permitted-cross-domain-policies: none
2022-09-29 23:17:13.136 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  referrer-policy: strict-origin-when-cross-origin
2022-09-29 23:17:13.136 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cache-control: public, max-age=300
2022-09-29 23:17:13.136 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-allow-origin: *
2022-09-29 23:17:13.136 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-allow-methods: POST, PUT, DELETE, GET, OPTIONS
2022-09-29 23:17:13.136 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-request-method: *
2022-09-29 23:17:13.136 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Authorization
2022-09-29 23:17:13.136 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-expose-headers: link, per-page, total
2022-09-29 23:17:13.137 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  vary: Accept-Encoding, Origin
2022-09-29 23:17:13.137 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  etag: W/"ec6e3f0e56f6a8daca4ec5b67e706c09"
2022-09-29 23:17:13.137 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-request-id: 9dff4cb7-7c9b-4eeb-a8a2-057c6000a0dd
2022-09-29 23:17:13.137 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-runtime: 0.069083
2022-09-29 23:17:13.137 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  alternate-protocol: 443:npn-spdy/2
2022-09-29 23:17:13.137 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-cache-status: HIT
2022-09-29 23:17:13.138 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  age: 7
2022-09-29 23:17:13.138 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  expires: Thu, 29 Sep 2022 16:22:12 GMT
2022-09-29 23:17:13.138 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  server: cloudflare
2022-09-29 23:17:13.138 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-ray: 75260716cbd0e68e-HKG
2022-09-29 23:17:13.144 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  {"id":"ethereum","symbol":"eth","name":"Ethereum","asset_platform_id":null,"platforms":{"":""},"detail_platforms":{"":{"decimal_place":null,"contract_address":""}},"block_time_in_minutes":0,"hashing_algorithm":"Ethash","categories":["Smart Contract Platform","Ethereum Ecosystem"],"public_notice":null,"additional_notices":[],"description":{"en":"Ethereum is a \u003ca href=\"https://www.coingecko.com/en?category_id=29\u0026view=market\"\u003esmart contract platform\u003c/a\u003e that enables developers to build tokens and decentralized applications (dapps). ETH is the native currency for the Ethereum platform and also works as the transaction fees to miners on the Ethereum network.\r\n\r\nEthereum is the pioneer for blockchain based smart contracts. Smart contract is essentially a computer code that runs exactly as programmed without any possibility of downtime, censorship, fraud or third-party interference. It can facilitate the exchange of money, content, property, shares, or anything of value. When running on the blockchain a smart contract becomes like a self-operating computer program that automatically executes when specific conditions are met.\r\n\r\nEthereum allows programmers to run complete-turing smart contracts that is capable of any customizations. Rather than giving a set of limited operations, Ethereum allows developers to have complete control over customization of their smart contract, giving developers the power to build unique and innovative applications.\r\n\r\nEthereum being the first blockchain based smart contract platform, they have gained much popularity, resulting in new competitors fighting for market share. The competitors includes: \u003ca href=\"https://www.coingecko.com/en/coins/ethereum_classic\"\u003eEthereum Classic\u003c/a\u003e which is the oldchain of Ethereum, \u003ca href=\"https://www.coingecko.com/en/coins/qtum\"\u003eQtum\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/eos\"\u003eEOS\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/neo\"\u003eNeo\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/icon\"\u003eIcon\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/tron\"\u003eTron\u003c/a\u003e and \u003ca href=\"https://www.coingecko.com/en/coins/cardano\"\u003eCardano\u003c/a\u003e.\r\n\r\nEthereum wallets are fairly simple to set up with multiple popular choices such as myetherwallet, \u003ca href=\"https://www.coingecko.com/buzz/complete-beginners-guide-to-metamask?locale=en\"\u003emetamask\u003c/a\u003e, and \u003ca href=\"https://www.coingecko.com/buzz/trezor-model-t-wallet-review\"\u003eTrezor\u003c/a\u003e. Read here for more guide on using ethereum wallet: \u003ca href=\"https://www.coingecko.com/buzz/how-to-use-an-ethereum-wallet\"\u003eHow to Use an Ethereum Wallet\u003c/a\u003e"},"links":{"homepage":["https://www.ethereum.org/","",""],"blockchain_site":["https://etherscan.io/","https://ethplorer.io/","https://blockchair.com/ethereum","https://eth.tokenview.io/","","","","","",""],"official_forum_url":["https://forum.ethereum.org/","",""],"chat_url":["","https://discord.com/invite/CetY6Y4",""],"announcement_url":["",""],"twitter_screen_name":"ethereum","facebook_username":"ethereumproject","bitcointalk_thread_identifier":428589,"telegram_channel_identifier":"","subreddit_url":"https://www.reddit.com/r/ethereum","repos_url":{"github":["https://github.com/ethereum/go-ethereum","https://github.com/ethereum/py-evm","https://github.com/ethereum/aleth","https://github.com/ethereum/web3.py","https://github.com/ethereum/solidity","https://github.com/ethereum/sharding","https://github.com/ethereum/casper","https://github.com/paritytech/parity"],"bitbucket":[]}},"image":{"thumb":"https://assets.coingecko.com/coins/images/279/thumb/ethereum.png?1595348880","small":"https://assets.coingecko.com/coins/images/279/small/ethereum.png?1595348880","large":"https://assets.coingecko.com/coins/images/279/large/ethereum.png?1595348880"},"c
2022-09-29 23:17:13.144 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  ountry_origin":"","genesis_date":"2015-07-30","sentiment_votes_up_percentage":69.53,"sentiment_votes_down_percentage":30.47,"ico_data":{"ico_start_date":"2014-07-20T00:00:00.000Z","ico_end_date":"2014-09-01T00:00:00.000Z","short_desc":"A decentralized platform for applications","description":null,"links":{},"softcap_currency":"","hardcap_currency":"","total_raised_currency":"","softcap_amount":null,"hardcap_amount":null,"total_raised":null,"quote_pre_sale_currency":"","base_pre_sale_amount":null,"quote_pre_sale_amount":null,"quote_public_sale_currency":"BTC","base_public_sale_amount":1.0,"quote_public_sale_amount":0.00074794,"accepting_currencies":"","country_origin":"","pre_sale_start_date":null,"pre_sale_end_date":null,"whitelist_url":"","whitelist_start_date":null,"whitelist_end_date":null,"bounty_detail_url":"","amount_for_sale":null,"kyc_required":true,"whitelist_available":null,"pre_sale_available":null,"pre_sale_ended":false},"market_cap_rank":2,"coingecko_rank":2,"coingecko_score":78.783,"developer_score":97.494,"community_score":72.746,"liquidity_score":95.183,"public_interest_score":0.236,"market_data":{"current_price":{"aed":4872.54,"ars":195115,"aud":2040.43,"bch":11.425832,"bdt":134319,"bhd":500.63,"bmd":1326.57,"bnb":4.706855,"brl":7147.94,"btc":0.06878001,"cad":1815.07,"chf":1298.33,"clp":1285510,"cny":9451.79,"czk":33259,"dkk":10076.71,"dot":208.407,"eos":1108,"eth":1.0,"eur":1355.02,"gbp":1195.76,"hkd":10413.48,"huf":569452,"idr":20153811,"ils":4728.02,"inr":108128,"jpy":191426,"krw":1892848,"kwd":411.3,"lkr":477052,"ltc":24.895338,"mmk":2786503,"mxn":26695,"myr":6149.3,"ngn":573130,"nok":14196.71,"nzd":2320.57,"php":77667,"pkr":307697,"pln":6577.72,"rub":77438,"sar":4985.41,"sek":14815.17,"sgd":1902.16,"thb":50284,"try":24527,"twd":42037,"uah":48766,"usd":1326.57,"vef":132.83,"vnd":31517393,"xag":70.37,"xau":0.797466,"xdr":1047.35,"xlm":12274,"xrp":3054,"yfi":0.16356957,"zar":23838,"bits":68780,"link":169.959,"sats":6878001},"total_value_locked":null,"mcap_to_tvl_ratio":null,"fdv_to_tvl_ratio":null,"roi":{"times":90.95926735667645,"currency":"btc","percentage":9095.926735667645},"ath":{"aed":17918.33,"ars":488484,"aud":6693.75,"bch":14.621519,"bdt":418411,"bhd":1839.17,"bmd":4878.26,"bnb":7747,"brl":26931,"btc":0.1474984,"cad":6108.18,"chf":4452.99,"clp":3990223,"cny":31155,"czk":107973,"dkk":31448,"dot":258.737,"eos":1566,"eth":1.003981,"eur":4228.93,"gbp":3610.11,"hkd":38002,"huf":1542639,"idr":69531233,"ils":15168.65,"inr":362338,"jpy":553787,"krw":5790811,"kwd":1471.83,"lkr":1064769,"ltc":32.998587,"mmk":8917781,"mxn":102241,"myr":20262,"ngn":2008213,"nok":43275,"nzd":6993.89,"php":244091,"pkr":839917,"pln":19645.56,"rub":403838,"sar":18294.87,"sek":43214,"sgd":6489.66,"thb":161550,"try":70272,"twd":135247,"uah":130195,"usd":4878.26,"vef":283483138,"vnd":110456068,"xag":211.07,"xau":2.69,"xdr":3455.91,"xlm":17299,"xrp":5418,"yfi":0.29805191,"zar":75832,"bits":147498,"link":2891,"sats":14749840},"ath_change_percentage":{"aed":-72.73853,"ars":-59.9522,"aud":-69.40287,"bch":-21.45468,"bdt":-67.8171,"bhd":-72.71079,"bmd":-72.73813,"bnb":-99.93914,"brl":-73.37054,"btc":-53.31246,"cad":-70.18366,"chf":-70.74673,"clp":-67.48306,"cny":-69.58523,"czk":-69.06942,"dkk":-67.84388,"dot":-19.27605,"eos":-29.26984,"eth":-0.39652,"eur":-67.84399,"gbp":-66.66031,"hkd":-72.52899,"huf":-62.89324,"idr":-70.95717,"ils":-68.75187,"inr":-70.06154,"jpy":-65.30831,"krw":-67.22168,"kwd":-71.98491,"lkr":-55.08393,"ltc":-24.5359,"mmk":-68.67479,"mxn":-73.79794,"myr":-69.57447,"ngn":-71.3889,"nok":-67.06842,"nzd":-66.69609,"php":-68.12532,"pkr":-63.27359,"pln":-66.37719,"rub":-80.77615,"sar":-72.68068,"sek":-65.5799,"sgd":-70.60763,"thb":-68.76354,"try":-64.98058,"twd":-68.82483,"uah":-62.44969,"usd":-72.73813,"vef":-99.99995,"vnd":-71.39434,"xag":-66.48968,"xau":-70.2192,"xdr":-69.61758,"xlm":-28.87915,"xrp":-43.60829,"yfi":-44.85668,"zar":-68.47666,"bits":-53.31246,"link":-94.08308,"sats":-53.31246},"ath_date":{"aed":"2021-11-10T14
2022-09-29 23:17:13.144 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  :24:19.604Z","ars":"2021-11-10T14:24:19.604Z","aud":"2021-12-01T08:38:24.623Z","bch":"2022-08-19T08:40:54.729Z","bdt":"2021-11-10T14:24:19.604Z","bhd":"2021-11-10T14:24:19.604Z","bmd":"2021-11-10T14:24:19.604Z","bnb":"2017-10-19T00:00:00.000Z","brl":"2021-11-09T04:03:16.303Z","btc":"2017-06-12T00:00:00.000Z","cad":"2021-12-01T08:38:24.623Z","chf":"2021-11-10T16:19:02.090Z","clp":"2021-12-01T16:19:01.053Z","cny":"2021-11-10T14:24:19.604Z","czk":"2021-12-01T08:38:24.623Z","dkk":"2021-12-01T08:38:24.623Z","dot":"2022-05-11T12:59:46.918Z","eos":"2022-05-12T07:29:33.469Z","eth":"2019-10-15T16:26:30.106Z","eur":"2021-12-01T08:38:24.623Z","gbp":"2021-11-12T02:17:36.494Z","hkd":"2021-11-10T14:24:19.604Z","huf":"2021-12-01T08:38:24.623Z","idr":"2021-11-10T14:24:19.604Z","ils":"2021-11-10T14:24:19.604Z","inr":"2021-11-10T14:24:19.604Z","jpy":"2021-11-10T16:19:02.090Z","krw":"2021-11-08T23:48:45.609Z","kwd":"2021-11-10T14:24:19.604Z","lkr":"2022-04-03T22:43:06.993Z","ltc":"2022-05-12T05:31:14.227Z","mmk":"2021-11-10T14:24:19.604Z","mxn":"2021-12-01T08:38:24.623Z","myr":"2021-11-10T14:24:19.604Z","ngn":"2021-11-09T04:03:16.303Z","nok":"2021-12-01T08:38:24.623Z","nzd":"2021-12-01T08:38:24.623Z","php":"2021-11-10T14:24:19.604Z","pkr":"2021-12-01T08:38:24.623Z","pln":"2021-12-01T08:38:24.623Z","rub":"2022-03-07T16:44:57.811Z","sar":"2021-11-10T14:24:19.604Z","sek":"2021-12-01T08:38:24.623Z","sgd":"2021-11-09T00:00:00.000Z","thb":"2021-12-01T08:38:24.623Z","try":"2021-12-20T16:43:34.305Z","twd":"2021-11-10T14:24:19.604Z","uah":"2021-12-01T08:38:24.623Z","usd":"2021-11-10T14:24:19.604Z","vef":"2021-01-04T06:47:56.457Z","vnd":"2021-11-10T14:24:19.604Z","xag":"2021-12-01T16:19:01.053Z","xau":"2021-12-01T08:38:24.623Z","xdr":"2021-11-10T14:24:19.604Z","xlm":"2022-05-12T05:31:14.227Z","xrp":"2021-12-04T07:02:03.517Z","yfi":"2020-07-19T00:00:00.000Z","zar":"2021-12-01T08:38:24.623Z","bits":"2017-06-12T00:00:00.000Z","link":"2017-11-29T00:00:00.000Z","sats":"2017-06-12T00:00:00.000Z"},"atl":{"aed":1.59,"ars":4.1,"aud":0.596152,"bch":0.20411961,"bdt":33.7,"bhd":0.163288,"bmd":0.432979,"bnb":3.541506,"brl":1.69,"btc":0.00160204,"cad":0.561954,"chf":0.413801,"clp":297.03,"cny":2.81,"czk":10.33,"dkk":2.84,"dot":40.15685,"eos":30.013555,"eth":0.99987722,"eur":0.381455,"gbp":0.280314,"hkd":3.35,"huf":118.42,"idr":5876.48,"ils":1.67,"inr":28.13,"jpy":51.85,"krw":489.4,"kwd":0.130639,"lkr":61.02,"ltc":0.13982924,"mmk":468.1,"mxn":7.16,"myr":1.85,"ngn":86.13,"nok":3.52,"nzd":0.641302,"php":20.05,"pkr":45.2,"pln":1.62,"rub":26.88,"sar":1.62,"sek":3.59,"sgd":0.601615,"thb":15.33,"try":1.26,"twd":14.02,"uah":9.54,"usd":0.432979,"vef":2.74,"vnd":9662.0,"xag":0.02721328,"xau":0.00036799,"xdr":0.306291,"xlm":664.668,"xrp":286.238,"yfi":0.00878891,"zar":5.74,"bits":7718.95,"link":21.519699,"sats":771895},"atl_change_percentage":{"aed":307052.05223,"ars":4766156.7308,"aud":343452.62322,"bch":5526.36744,"bdt":399467.46285,"bhd":307267.48612,"bmd":307052.47035,"bnb":33.13695,"brl":423879.81334,"btc":4198.47019,"cad":323990.01558,"chf":314699.71268,"clp":436728.6508,"cny":336786.7412,"czk":323151.18247,"dkk":355366.4476,"dot":420.11781,"eos":3589.45275,"eth":0.01228,"eur":356391.25289,"gbp":429275.06027,"hkd":311114.13213,"huf":483302.40632,"idr":343538.42758,"ils":284086.72882,"inr":385515.90264,"jpy":370460.77349,"krw":387745.09678,"kwd":315529.36994,"lkr":783642.79322,"ltc":17708.92732,"mmk":596681.32407,"mxn":374218.4574,"myr":333118.24429,"ngn":666966.22945,"nok":405242.11284,"nzd":363104.44575,"php":388006.4432,"pkr":682432.93371,"pln":406632.69119,"rub":288701.00391,"sar":307708.92693,"sek":413771.36058,"sgd":316957.32927,"thb":329084.66661,"try":1959879.65442,"twd":300649.74578,"uah":512118.41436,"usd":307052.47035,"vef":4765.74341,"vnd":326920.30986,"xag":259810.62684,"xau":217425.1203,"xdr":342708.07101,"xlm":1751.0144,"xrp":967.43494,"yfi":1770.03507,"zar":416179.38274,"bits":792.1343,"link":694.79729,"sats":792.1343},"atl_date":{"aed":"2015-10-20T00:00:00.0
2022-09-29 23:17:13.144 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  00Z","ars":"2015-10-20T00:00:00.000Z","aud":"2015-10-20T00:00:00.000Z","bch":"2017-11-12T00:00:00.000Z","bdt":"2015-10-20T00:00:00.000Z","bhd":"2015-10-20T00:00:00.000Z","bmd":"2015-10-20T00:00:00.000Z","bnb":"2021-04-12T21:29:59.636Z","brl":"2015-10-20T00:00:00.000Z","btc":"2015-10-20T00:00:00.000Z","cad":"2015-10-20T00:00:00.000Z","chf":"2015-10-20T00:00:00.000Z","clp":"2015-10-20T00:00:00.000Z","cny":"2015-10-20T00:00:00.000Z","czk":"2015-10-20T00:00:00.000Z","dkk":"2015-10-20T00:00:00.000Z","dot":"2021-03-03T00:43:22.672Z","eos":"2019-04-16T00:00:00.000Z","eth":"2019-10-22T00:00:00.000Z","eur":"2015-10-20T00:00:00.000Z","gbp":"2015-10-20T00:00:00.000Z","hkd":"2015-10-20T00:00:00.000Z","huf":"2015-10-20T00:00:00.000Z","idr":"2015-10-20T00:00:00.000Z","ils":"2015-10-20T00:00:00.000Z","inr":"2015-10-20T00:00:00.000Z","jpy":"2015-10-20T00:00:00.000Z","krw":"2015-10-20T00:00:00.000Z","kwd":"2015-10-20T00:00:00.000Z","lkr":"2015-10-20T00:00:00.000Z","ltc":"2015-10-20T00:00:00.000Z","mmk":"2015-10-20T00:00:00.000Z","mxn":"2015-10-20T00:00:00.000Z","myr":"2015-10-20T00:00:00.000Z","ngn":"2015-10-20T00:00:00.000Z","nok":"2015-10-20T00:00:00.000Z","nzd":"2015-10-20T00:00:00.000Z","php":"2015-10-20T00:00:00.000Z","pkr":"2015-10-20T00:00:00.000Z","pln":"2015-10-20T00:00:00.000Z","rub":"2015-10-20T00:00:00.000Z","sar":"2015-10-20T00:00:00.000Z","sek":"2015-10-20T00:00:00.000Z","sgd":"2015-10-20T00:00:00.000Z","thb":"2015-10-20T00:00:00.000Z","try":"2015-10-20T00:00:00.000Z","twd":"2015-10-20T00:00:00.000Z","uah":"2015-10-20T00:00:00.000Z","usd":"2015-10-20T00:00:00.000Z","vef":"2015-10-20T00:00:00.000Z","vnd":"2015-10-20T00:00:00.000Z","xag":"2015-10-20T00:00:00.000Z","xau":"2015-10-20T00:00:00.000Z","xdr":"2015-10-20T00:00:00.000Z","xlm":"2018-11-20T00:00:00.000Z","xrp":"2018-12-20T00:00:00.000Z","yfi":"2020-09-14T08:23:43.987Z","zar":"2015-10-20T00:00:00.000Z","bits":"2016-12-27T00:00:00.000Z","link":"2020-08-16T08:12:39.777Z","sats":"2016-12-27T00:00:00.000Z"},"market_cap":{"aed":588678312735,"ars":23575211784337,"aud":246789260237,"bch":1379649612,"bdt":16227787246549,"bhd":60484392955,"bmd":160269834085,"bnb":568709409,"brl":862956894645,"btc":8304975,"cad":219432161178,"chf":156945356916,"clp":155437698586916,"cny":1141922567853,"czk":4023894564091,"dkk":1218932223130,"dot":25173742292,"eos":133894394240,"eth":120734247,"eur":163880392907,"gbp":144890020266,"hkd":1258110184072,"huf":68922296809574,"idr":2433449012331294,"ils":571218517010,"inr":13072480016644,"jpy":23143865559628,"krw":228839504564108,"kwd":49691341518,"lkr":57635272268011,"ltc":3004117328,"mmk":336652713753904,"mxn":3226300515881,"myr":742930815899,"ngn":69242979117896,"nok":1716570859312,"nzd":280694984717,"php":9376585681499,"pkr":37174588015915,"pln":795311324963,"rub":9355734576084,"sar":602350451471,"sek":1791957782519,"sgd":229867971155,"thb":6073263810642,"try":2963693744908,"twd":5078390257990,"uah":5891659657593,"usd":160269834085,"vef":16047818487,"vnd":3807789742661065,"xag":8531799540,"xau":96469619,"xdr":126536559946,"xlm":1482763172064,"xrp":368688066064,"yfi":19809044,"zar":2879503039445,"bits":8304975286652,"link":20504464031,"sats":830497528665188},"market_cap_rank":2,"fully_diluted_valuation":{},"total_volume":{"aed":46417249173,"ars":1858723644318,"aud":19437771627,"bch":108845852,"bdt":1279560037881,"bhd":4769190708,"bmd":12637266675,"bnb":44838888,"brl":68093384022,"btc":655219,"cad":17290902216,"chf":12368257179,"clp":12246143270965,"cny":90040525056,"czk":316835193519,"dkk":95993713916,"dot":1985350519,"eos":10555801665,"eth":9529483,"eur":12908285496,"gbp":11391168994,"hkd":99201911532,"huf":5424767543645,"idr":191991153638799,"ils":45040545341,"inr":1030054659457,"jpy":1823579345290,"krw":18031831188747,"kwd":3918159258,"lkr":4544537714604,"ltc":237160338,"mmk":26545046013909,"mxn":254300349155,"myr":58580049670,"ngn":5459804694069,"nok":135242031263,"nzd":22106459055,"php":739874051994,"pkr":2931214005160,"pln":62661254942,"rub":7376991
2022-09-29 23:17:13.144 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  02576,"sar":47492528919,"sek":141133714545,"sgd":18120576685,"thb":479020420734,"try":233652003204,"twd":400456037653,"uah":464557005839,"usd":12637266675,"vef":1265369512,"vnd":300243989727995,"xag":670323970,"xau":7596893,"xdr":9977400059,"xlm":116925205829,"xrp":29089515792,"yfi":1558212,"zar":227086210205,"bits":655218716934,"link":1619080093,"sats":65521871693437},"high_24h":{"aed":4973.63,"ars":198868,"aud":2079.05,"bch":11.773638,"bdt":139087,"bhd":510.86,"bmd":1354.09,"bnb":4.792134,"brl":7279.84,"btc":0.06907022,"cad":1842.82,"chf":1320.82,"clp":1299605,"cny":9750.09,"czk":34349,"dkk":10366.43,"dot":210.325,"eos":1169,"eth":1.0,"eur":1393.81,"gbp":1249.69,"hkd":10628.26,"huf":580411,"idr":20548588,"ils":4780.86,"inr":110198,"jpy":195143,"krw":1933290,"kwd":420.06,"lkr":493782,"ltc":25.245497,"mmk":2848763,"mxn":27293,"myr":6258.42,"ngn":585074,"nok":14474.4,"nzd":2367.99,"php":79524,"pkr":316905,"pln":6709.72,"rub":79146,"sar":5100.48,"sek":15319.23,"sgd":1941.82,"thb":51237,"try":25063,"twd":42950,"uah":49857,"usd":1354.09,"vef":135.58,"vnd":32144471,"xag":71.82,"xau":0.815864,"xdr":1054.28,"xlm":12375,"xrp":3563,"yfi":0.16431094,"zar":24207,"bits":69070,"link":174.399,"sats":6907022},"low_24h":{"aed":4756.89,"ars":190525,"aud":2001.24,"bch":11.270123,"bdt":131131,"bhd":488.57,"bmd":1295.08,"bnb":4.641298,"brl":7006.76,"btc":0.06765578,"cad":1778.3,"chf":1270.07,"clp":1249671,"cny":9236.0,"czk":32695,"dkk":9889.95,"dot":205.119,"eos":1094,"eth":1.0,"eur":1329.98,"gbp":1178.27,"hkd":10166.37,"huf":558679,"idr":19680351,"ils":4604.82,"inr":105840,"jpy":187224,"krw":1854710,"kwd":401.75,"lkr":465729,"ltc":24.611387,"mmk":2720365,"mxn":26217,"myr":6003.35,"ngn":559527,"nok":13957.63,"nzd":2280.63,"php":75832,"pkr":296591,"pln":6462.57,"rub":75699,"sar":4867.34,"sek":14569.39,"sgd":1861.3,"thb":49310,"try":24003,"twd":41080,"uah":47608,"usd":1295.08,"vef":129.68,"vnd":30769334,"xag":69.38,"xau":0.783361,"xdr":989.33,"xlm":12068,"xrp":2854,"yfi":0.16060881,"zar":23366,"bits":67656,"link":164.068,"sats":6765578},"price_change_24h":-11.83334803686,"price_change_percentage_24h":-0.88414,"price_change_percentage_7d":5.8903,"price_change_percentage_14d":-18.88268,"price_change_percentage_30d":-14.69997,"price_change_percentage_60d":-21.81881,"price_change_percentage_200d":-48.57188,"price_change_percentage_1y":-52.60542,"market_cap_change_24h":-1883074944.348,"market_cap_change_percentage_24h":-1.1613,"price_change_24h_in_currency":{"aed":-43.477803841336936,"ars":-1452.7839606594061,"aud":-14.598266301851709,"bch":-0.3191656074238942,"bdt":-3157.0139196971722,"bhd":-4.278423030910233,"bmd":-11.833348036858979,"bnb":-0.062036729769069154,"brl":-21.468185148735756,"btc":0.00050147,"cad":-10.550924807533647,"chf":-7.746935187440158,"clp":1180.78,"cny":-185.36181450669392,"czk":-763.2339891952579,"dkk":-190.9483373677158,"dot":0.90134339,"eos":-51.45455859366416,"eth":0.0,"eur":-25.629058524687025,"gbp":-38.60026984729461,"hkd":-92.72121360557139,"huf":-1025.5597266132245,"idr":-199184.8722877875,"ils":2.54,"inr":-827.9574713049369,"jpy":-1528.375593691453,"krw":-14148.954239486251,"kwd":-4.00618272465573,"lkr":-11010.334757892007,"ltc":-0.003651073281083939,"mmk":-29260.831142453942,"mxn":-460.7003340759329,"myr":-22.062681662899195,"ngn":-4162.215594212059,"nok":-138.02840611977444,"nzd":-24.43831011478096,"php":-1059.5340164771624,"pkr":-9169.065696637379,"pln":-64.32445151813408,"rub":-791.2949942103587,"sar":-55.80486805068995,"sek":-202.9515186032604,"sgd":-20.609624501224516,"thb":-494.21111327944527,"try":-254.08855442590357,"twd":-410.44598771923484,"uah":-513.9471064935788,"usd":-11.833348036858979,"vef":-1.184873138930783,"vnd":-254720.7004112266,"xag":-0.9398096623575327,"xau":-0.009429049395634716,"xdr":43.89,"xlm":124.277,"xrp":12.002075,"yfi":0.00175685,"zar":-43.57745128017268,"bits":501.47,"link":5.890609,"sats":50147},"price_change_percentage_1h_in_currency":{"aed":0.15089,"ars":0.13778,"aud":-0.0759,"bch":0.1328
2022-09-29 23:17:13.144 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  1,"bdt":0.15089,"bhd":0.15381,"bmd":0.15089,"bnb":0.31886,"brl":0.02651,"btc":-0.19168,"cad":0.04939,"chf":0.10046,"clp":0.64943,"cny":0.14245,"czk":-0.16392,"dkk":-0.00409,"dot":-0.12635,"eos":0.3329,"eth":0.0,"eur":-0.00672,"gbp":-0.31459,"hkd":0.15242,"huf":-0.10193,"idr":0.32487,"ils":0.15089,"inr":-0.0536,"jpy":-0.00722,"krw":-0.1415,"kwd":0.16155,"lkr":0.15089,"ltc":0.1908,"mmk":0.15089,"mxn":-0.13175,"myr":0.15089,"ngn":0.15089,"nok":-0.01224,"nzd":-0.13558,"php":0.16542,"pkr":0.15089,"pln":-0.09693,"rub":0.09498,"sar":0.14439,"sek":-0.06877,"sgd":0.07314,"thb":-0.07745,"try":-0.02998,"twd":0.09719,"uah":0.15089,"usd":0.15089,"vef":0.15089,"vnd":0.15089,"xag":-0.33241,"xau":-0.1961,"xdr":0.15089,"xlm":0.14991,"xrp":0.1204,"yfi":-0.3891,"zar":-0.0947,"bits":-0.19168,"link":0.64488,"sats":-0.19168},"price_change_percentage_24h_in_currency":{"aed":-0.88441,"ars":-0.73908,"aud":-0.71037,"bch":-2.71746,"bdt":-2.29641,"bhd":-0.84736,"bmd":-0.88414,"bnb":-1.30086,"brl":-0.29944,"btc":0.73445,"cad":-0.57794,"chf":-0.59315,"clp":0.09194,"cny":-1.92341,"czk":-2.24334,"dkk":-1.85971,"dot":0.43437,"eos":-4.43756,"eth":0.0,"eur":-1.85631,"gbp":-3.12715,"hkd":-0.88254,"huf":-0.17977,"idr":-0.97865,"ils":0.05376,"inr":-0.7599,"jpy":-0.79209,"krw":-0.74195,"kwd":-0.96463,"lkr":-2.25593,"ltc":-0.01466,"mmk":-1.03918,"mxn":-1.69654,"myr":-0.3575,"ngn":-0.72099,"nok":-0.96289,"nzd":-1.04214,"php":-1.34585,"pkr":-2.89367,"pln":-0.96844,"rub":-1.0115,"sar":-1.10697,"sek":-1.35138,"sgd":-1.07187,"thb":-0.97327,"try":-1.02533,"twd":-0.96695,"uah":-1.04292,"usd":-0.88414,"vef":-0.88414,"vnd":-0.80171,"xag":-1.31801,"xau":-1.16856,"xdr":4.37436,"xlm":1.02288,"xrp":0.3946,"yfi":1.08573,"zar":-0.18247,"bits":0.73445,"link":3.59034,"sats":0.73445},"price_change_percentage_7d_in_currency":{"aed":5.89117,"ars":7.4885,"aud":7.67206,"bch":-0.43697,"bdt":3.46304,"bhd":5.99253,"bmd":5.8903,"bnb":-0.5958,"brl":10.32091,"btc":1.76262,"cad":7.4933,"chf":7.13271,"clp":9.41023,"cny":7.03198,"czk":5.87044,"dkk":6.32678,"dot":2.34496,"eos":5.28387,"eth":0.0,"eur":6.34653,"gbp":7.42929,"hkd":5.89273,"huf":9.75554,"idr":6.82874,"ils":9.03884,"inr":7.81945,"jpy":5.88937,"krw":8.09616,"kwd":6.14982,"lkr":5.99713,"ltc":1.7576,"mmk":5.83582,"mxn":6.35001,"myr":7.82087,"ngn":6.42484,"nok":9.29245,"nzd":8.0103,"php":6.83831,"pkr":2.37046,"pln":7.98091,"rub":0.34636,"sar":5.78002,"sek":6.57453,"sgd":7.02044,"thb":7.81124,"try":6.74316,"twd":6.67636,"uah":5.32161,"usd":5.8903,"vef":5.8903,"vnd":6.16706,"xag":9.30238,"xau":6.07558,"xdr":12.62584,"xlm":8.847,"xrp":-2.86754,"yfi":5.04499,"zar":7.2203,"bits":1.76262,"link":-8.96495,"sats":1.76262},"price_change_percentage_14d_in_currency":{"aed":-18.88187,"ars":-16.48117,"aud":-15.74679,"bch":-15.978,"bdt":-15.23543,"bhd":-18.79188,"bmd":-18.88268,"bnb":-19.53223,"brl":-15.36425,"btc":-14.82472,"cad":-15.66593,"chf":-17.47531,"clp":-14.59967,"cny":-16.9835,"czk":-17.08519,"dkk":-17.29433,"dot":-8.14936,"eos":0.04398,"eth":0.0,"eur":-17.29167,"gbp":-15.59388,"hkd":-18.87132,"huf":-14.32045,"idr":-17.41521,"ils":-15.60021,"inr":-16.78442,"jpy":-18.08262,"krw":-16.82335,"kwd":-18.55169,"lkr":-20.08369,"ltc":-8.56072,"mmk":-18.86633,"mxn":-18.24812,"myr":-16.93852,"ngn":-18.00487,"nok":-13.97995,"nzd":-14.63813,"php":-16.81976,"pkr":-19.88676,"pln":-14.89079,"rub":-20.71638,"sar":-18.92107,"sek":-15.25789,"sgd":-17.24537,"thb":-16.10415,"try":-17.82667,"twd":-17.29373,"uah":-19.26121,"usd":-18.88268,"vef":-18.88268,"vnd":-18.34306,"xag":-15.4673,"xau":-17.20657,"xdr":-10.02264,"xlm":-20.98016,"xrp":-36.06806,"yfi":-7.84499,"zar":-16.64634,"bits":-14.82472,"link":-22.57874,"sats":-14.82472},"price_change_percentage_30d_in_currency":{"aed":-14.69886,"ars":-9.29081,"aud":-9.38001,"bch":-12.08887,"bdt":-9.15403,"bhd":-14.61966,"bmd":-14.69997,"bnb":-13.31736,"brl":-8.60019,"btc":-10.15348,"cad":-10.24677,"chf":-13.69782,"clp":-6.33535,"cny":-12.02046,"czk":-12.96282,"dkk":-12.80149,"dot":-2.33077,"eos":6.55753,"eth":0.0,"eur":-12.7
2022-09-29 23:17:13.144 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  8778,"gbp":-9.88107,"hkd":-14.67068,"huf":-10.11393,"idr":-12.95091,"ils":-8.45443,"inr":-13.05607,"jpy":-11.18658,"krw":-9.57923,"kwd":-14.17068,"lkr":-14.59431,"ltc":-10.61162,"mmk":-14.72084,"mxn":-14.16701,"myr":-11.83762,"ngn":-13.54158,"nok":-6.25046,"nzd":-8.09767,"php":-11.2167,"pkr":-10.96045,"pln":-10.58487,"rub":-19.36226,"sar":-14.64571,"sek":-10.60521,"sgd":-12.34492,"thb":-11.02559,"try":-13.27485,"twd":-11.03479,"uah":-15.13634,"usd":-14.69997,"vef":-14.69997,"vnd":-13.52452,"xag":-14.96479,"xau":-10.79741,"xdr":-8.84534,"xlm":-16.30885,"xrp":-34.37399,"yfi":-4.41311,"zar":-9.02614,"bits":-10.15348,"link":-26.57409,"sats":-10.15348},"price_change_percentage_60d_in_currency":{"aed":-21.81998,"ars":-12.35827,"aud":-15.91599,"bch":-2.60469,"bdt":-16.51183,"bhd":-21.70076,"bmd":-21.81881,"bnb":-20.28887,"brl":-18.56176,"btc":-4.15615,"cad":-16.39774,"chf":-19.64033,"clp":-16.0519,"cny":-17.40444,"czk":-18.5794,"dkk":-18.41482,"dot":0.91078,"eos":-14.62723,"eth":0.0,"eur":-18.33505,"gbp":-14.20015,"hkd":-21.81901,"huf":-15.12995,"idr":-19.82906,"ils":-18.26299,"inr":-19.53414,"jpy":-15.30589,"krw":-14.40444,"kwd":-21.01301,"lkr":-21.58136,"ltc":-11.41509,"mmk":-11.43538,"mxn":-22.75524,"myr":-18.56895,"ngn":-18.77894,"nok":-13.37836,"nzd":-14.11993,"php":-17.3113,"pkr":-24.15905,"pln":-16.34399,"rub":-26.68565,"sar":-21.77689,"sek":-13.93475,"sgd":-18.79816,"thb":-18.2803,"try":-19.31242,"twd":-17.45152,"uah":-22.25601,"usd":-21.81881,"vef":-21.81881,"vnd":-20.41658,"xag":-15.60877,"xau":-16.9929,"xdr":-16.14998,"xlm":-13.64825,"xrp":-30.3295,"yfi":13.03559,"zar":-15.40529,"bits":-4.15615,"link":-21.74235,"sats":-4.15615},"price_change_percentage_200d_in_currency":{"aed":-48.57265,"ars":-30.47244,"aud":-42.3257,"bch":28.94234,"bdt":-39.38461,"bhd":-48.49122,"bmd":-48.57188,"bnb":-31.85766,"brl":-45.39702,"btc":3.89145,"cad":-44.78995,"chf":-46.14586,"clp":-38.16515,"cny":-42.20238,"czk":-43.97995,"dkk":-42.69987,"dot":46.79011,"eos":-15.29351,"eth":0.0,"eur":-42.66759,"gbp":-39.55733,"hkd":-48.43756,"huf":-36.98034,"idr":-45.46524,"ils":-43.61622,"inr":-45.38935,"jpy":-36.72699,"krw":-40.68278,"kwd":-47.48637,"lkr":-27.34246,"ltc":2.15097,"mmk":-39.13792,"mxn":-50.53518,"myr":-43.16485,"ngn":-46.56838,"nok":-38.89942,"nzd":-38.7661,"php":-42.52408,"pkr":-33.24705,"pln":-41.86557,"rub":-77.59619,"sar":-48.48128,"sek":-41.07731,"sgd":-45.84701,"thb":-41.4981,"try":-35.59816,"twd":-42.64215,"uah":-35.5807,"usd":-48.57188,"vef":-48.57188,"vnd":-46.59246,"xag":-29.5651,"xau":-38.5392,"xdr":-43.28904,"xlm":-11.69434,"xrp":-6.98716,"yfi":16.73474,"zar":-38.53348,"bits":3.89145,"link":-13.10095,"sats":3.89145},"price_change_percentage_1y_in_currency":{"aed":-52.60742,"ars":-29.28061,"aud":-47.20669,"bch":93.70257,"bdt":-43.97175,"bhd":-52.55753,"bmd":-52.60542,"bnb":-44.14495,"brl":-52.96593,"btc":1.3196,"cad":-48.8618,"chf":-50.08062,"clp":-42.71186,"cny":-47.72657,"czk":-45.54419,"dkk":-43.42289,"dot":95.9204,"eos":46.20441,"eth":0.0,"eur":-43.42738,"gbp":-42.15886,"hkd":-52.19988,"huf":-33.90685,"idr":-49.71313,"ils":-47.37833,"inr":-47.93707,"jpy":-38.68681,"krw":-43.06689,"kwd":-51.2237,"lkr":-14.67498,"ltc":25.14698,"mmk":-46.80559,"mxn":-53.10786,"myr":-47.47219,"ngn":-50.15515,"nok":-41.43798,"nzd":-42.31604,"php":-45.74827,"pkr":-35.40287,"pln":-40.76225,"rub":-62.03667,"sar":-52.5165,"sek":-39.39164,"sgd":-49.92534,"thb":-46.90977,"try":-1.25908,"twd":-45.88485,"uah":-34.49027,"usd":-52.60542,"vef":-52.60542,"vnd":-50.5109,"xag":-43.4973,"xau":-50.58407,"xdr":-47.01463,"xlm":12.88397,"xrp":-1.9942,"yfi":64.92974,"zar":-43.63153,"bits":1.3196,"link":34.20475,"sats":1.3196},"market_cap_change_24h_in_currency":{"aed":-6918240538.055542,"ars":-233103774284.54688,"aud":-2038918948.880371,"bch":-41826894.277243376,"bdt":-427990250037.25195,"bhd":-687954129.0587006,"bmd":-1883074944.3485718,"bnb":-9290299.869598627,"brl":-5112488550.455811,"btc":32229,"cad":-1693490723.7938843,"chf":-1156485986.4953308,"clp":-133423864514
2022-09-29 23:17:13.145 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  .71875,"cny":-25659453610.16577,"czk":-96158382254.00098,"dkk":-24021389201.534912,"dot":117658157,"eos":-6523338939.160385,"eth":14937,"eur":-3266339261.455475,"gbp":-4420702673.499939,"hkd":-14765828868.483643,"huf":-137978534204.25,"idr":-31758044868821.0,"ils":-1293948133.871704,"inr":-154179700498.3086,"jpy":-223233446412.83203,"krw":-2488678813820.9688,"kwd":-610761003.2442169,"lkr":-1495564826794.6875,"ltc":-6886112.174203873,"mmk":-4489086057831.3125,"mxn":-64191930869.57373,"myr":-4756247633.319702,"ngn":-698435133545.8906,"nok":-19126364707.588867,"nzd":-3293944804.0578613,"php":-165059345250.20312,"pkr":-1215113196680.4375,"pln":-8734414395.597778,"rub":-122103280961.75586,"sar":-8398642538.919312,"sek":-26638332559.09326,"sgd":-3065006971.0610046,"thb":-78804423529.52148,"try":-38632389078.48584,"twd":-69526628670.31934,"uah":-78787751444.24219,"usd":-1883074944.3485718,"vef":-188552294.17759514,"vnd":-41537962794006.5,"xag":-88792316.65958023,"xau":-1219401.5122876912,"xdr":4963227166,"xlm":12676465667,"xrp":758425865,"yfi":231218,"zar":-13122273455.515625,"bits":32228768819,"link":747262057,"sats":3222876881867},"market_cap_change_percentage_24h_in_currency":{"aed":-1.16156,"ars":-0.97909,"aud":-0.81941,"bch":-2.9425,"bdt":-2.56962,"bhd":-1.12462,"bmd":-1.1613,"bnb":-1.60732,"brl":-0.58895,"btc":0.38958,"cad":-0.76585,"chf":-0.73148,"clp":-0.08576,"cny":-2.19766,"czk":-2.33391,"dkk":-1.93261,"dot":0.46958,"eos":-4.64567,"eth":0.01237,"eur":-1.95417,"gbp":-2.96074,"hkd":-1.16004,"huf":-0.19979,"idr":-1.28825,"ils":-0.22601,"inr":-1.16567,"jpy":-0.95533,"krw":-1.07582,"kwd":-1.21419,"lkr":-2.52925,"ltc":-0.2287,"mmk":-1.3159,"mxn":-1.95083,"myr":-0.63613,"ngn":-0.9986,"nok":-1.10194,"nzd":-1.15988,"php":-1.72988,"pkr":-3.16521,"pln":-1.08631,"rub":-1.2883,"sar":-1.37514,"sek":-1.46477,"sgd":-1.31583,"thb":-1.28094,"try":-1.28675,"twd":-1.35058,"uah":-1.31963,"usd":-1.1613,"vef":-1.1613,"vnd":-1.0791,"xag":-1.03,"xau":-1.24825,"xdr":4.0825,"xlm":0.86229,"xrp":0.20613,"yfi":1.18102,"zar":-0.45365,"bits":0.38958,"link":3.78223,"sats":0.38958},"total_supply":120732090.871192,"max_supply":null,"circulating_supply":120734246.871192,"last_updated":"2022-09-29T16:15:38.516Z"},"public_interest_stats":{"alexa_rank":8793,"bing_matches":null},"status_updates":[],"last_updated":"2022-09-29T16:15:38.516Z"}
2022-09-29 23:17:13.145 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- END HTTP (30345-byte body)
2022-09-29 23:17:13.656 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- 200 https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false (846ms)
2022-09-29 23:17:13.656 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  date: Thu, 29 Sep 2022 16:17:13 GMT
2022-09-29 23:17:13.656 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  content-type: application/json; charset=utf-8
2022-09-29 23:17:13.657 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-frame-options: SAMEORIGIN
2022-09-29 23:17:13.657 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-xss-protection: 1; mode=block
2022-09-29 23:17:13.657 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-content-type-options: nosniff
2022-09-29 23:17:13.657 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-download-options: noopen
2022-09-29 23:17:13.657 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-permitted-cross-domain-policies: none
2022-09-29 23:17:13.657 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  referrer-policy: strict-origin-when-cross-origin
2022-09-29 23:17:13.657 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cache-control: public, max-age=300
2022-09-29 23:17:13.658 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-allow-origin: *
2022-09-29 23:17:13.658 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-allow-methods: POST, PUT, DELETE, GET, OPTIONS
2022-09-29 23:17:13.658 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-request-method: *
2022-09-29 23:17:13.658 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Authorization
2022-09-29 23:17:13.658 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-expose-headers: link, per-page, total
2022-09-29 23:17:13.659 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  vary: Accept-Encoding, Origin
2022-09-29 23:17:13.659 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  etag: W/"ec6e3f0e56f6a8daca4ec5b67e706c09"
2022-09-29 23:17:13.659 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-request-id: 9dff4cb7-7c9b-4eeb-a8a2-057c6000a0dd
2022-09-29 23:17:13.659 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-runtime: 0.069083
2022-09-29 23:17:13.659 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  alternate-protocol: 443:npn-spdy/2
2022-09-29 23:17:13.660 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-cache-status: HIT
2022-09-29 23:17:13.660 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  age: 8
2022-09-29 23:17:13.660 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  expires: Thu, 29 Sep 2022 16:22:13 GMT
2022-09-29 23:17:13.660 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  server: cloudflare
2022-09-29 23:17:13.660 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-ray: 75260718ef21e68e-HKG
2022-09-29 23:17:13.666 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  {"id":"ethereum","symbol":"eth","name":"Ethereum","asset_platform_id":null,"platforms":{"":""},"detail_platforms":{"":{"decimal_place":null,"contract_address":""}},"block_time_in_minutes":0,"hashing_algorithm":"Ethash","categories":["Smart Contract Platform","Ethereum Ecosystem"],"public_notice":null,"additional_notices":[],"description":{"en":"Ethereum is a \u003ca href=\"https://www.coingecko.com/en?category_id=29\u0026view=market\"\u003esmart contract platform\u003c/a\u003e that enables developers to build tokens and decentralized applications (dapps). ETH is the native currency for the Ethereum platform and also works as the transaction fees to miners on the Ethereum network.\r\n\r\nEthereum is the pioneer for blockchain based smart contracts. Smart contract is essentially a computer code that runs exactly as programmed without any possibility of downtime, censorship, fraud or third-party interference. It can facilitate the exchange of money, content, property, shares, or anything of value. When running on the blockchain a smart contract becomes like a self-operating computer program that automatically executes when specific conditions are met.\r\n\r\nEthereum allows programmers to run complete-turing smart contracts that is capable of any customizations. Rather than giving a set of limited operations, Ethereum allows developers to have complete control over customization of their smart contract, giving developers the power to build unique and innovative applications.\r\n\r\nEthereum being the first blockchain based smart contract platform, they have gained much popularity, resulting in new competitors fighting for market share. The competitors includes: \u003ca href=\"https://www.coingecko.com/en/coins/ethereum_classic\"\u003eEthereum Classic\u003c/a\u003e which is the oldchain of Ethereum, \u003ca href=\"https://www.coingecko.com/en/coins/qtum\"\u003eQtum\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/eos\"\u003eEOS\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/neo\"\u003eNeo\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/icon\"\u003eIcon\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/tron\"\u003eTron\u003c/a\u003e and \u003ca href=\"https://www.coingecko.com/en/coins/cardano\"\u003eCardano\u003c/a\u003e.\r\n\r\nEthereum wallets are fairly simple to set up with multiple popular choices such as myetherwallet, \u003ca href=\"https://www.coingecko.com/buzz/complete-beginners-guide-to-metamask?locale=en\"\u003emetamask\u003c/a\u003e, and \u003ca href=\"https://www.coingecko.com/buzz/trezor-model-t-wallet-review\"\u003eTrezor\u003c/a\u003e. Read here for more guide on using ethereum wallet: \u003ca href=\"https://www.coingecko.com/buzz/how-to-use-an-ethereum-wallet\"\u003eHow to Use an Ethereum Wallet\u003c/a\u003e"},"links":{"homepage":["https://www.ethereum.org/","",""],"blockchain_site":["https://etherscan.io/","https://ethplorer.io/","https://blockchair.com/ethereum","https://eth.tokenview.io/","","","","","",""],"official_forum_url":["https://forum.ethereum.org/","",""],"chat_url":["","https://discord.com/invite/CetY6Y4",""],"announcement_url":["",""],"twitter_screen_name":"ethereum","facebook_username":"ethereumproject","bitcointalk_thread_identifier":428589,"telegram_channel_identifier":"","subreddit_url":"https://www.reddit.com/r/ethereum","repos_url":{"github":["https://github.com/ethereum/go-ethereum","https://github.com/ethereum/py-evm","https://github.com/ethereum/aleth","https://github.com/ethereum/web3.py","https://github.com/ethereum/solidity","https://github.com/ethereum/sharding","https://github.com/ethereum/casper","https://github.com/paritytech/parity"],"bitbucket":[]}},"image":{"thumb":"https://assets.coingecko.com/coins/images/279/thumb/ethereum.png?1595348880","small":"https://assets.coingecko.com/coins/images/279/small/ethereum.png?1595348880","large":"https://assets.coingecko.com/coins/images/279/large/ethereum.png?1595348880"},"c
2022-09-29 23:17:13.666 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  ountry_origin":"","genesis_date":"2015-07-30","sentiment_votes_up_percentage":69.53,"sentiment_votes_down_percentage":30.47,"ico_data":{"ico_start_date":"2014-07-20T00:00:00.000Z","ico_end_date":"2014-09-01T00:00:00.000Z","short_desc":"A decentralized platform for applications","description":null,"links":{},"softcap_currency":"","hardcap_currency":"","total_raised_currency":"","softcap_amount":null,"hardcap_amount":null,"total_raised":null,"quote_pre_sale_currency":"","base_pre_sale_amount":null,"quote_pre_sale_amount":null,"quote_public_sale_currency":"BTC","base_public_sale_amount":1.0,"quote_public_sale_amount":0.00074794,"accepting_currencies":"","country_origin":"","pre_sale_start_date":null,"pre_sale_end_date":null,"whitelist_url":"","whitelist_start_date":null,"whitelist_end_date":null,"bounty_detail_url":"","amount_for_sale":null,"kyc_required":true,"whitelist_available":null,"pre_sale_available":null,"pre_sale_ended":false},"market_cap_rank":2,"coingecko_rank":2,"coingecko_score":78.783,"developer_score":97.494,"community_score":72.746,"liquidity_score":95.183,"public_interest_score":0.236,"market_data":{"current_price":{"aed":4872.54,"ars":195115,"aud":2040.43,"bch":11.425832,"bdt":134319,"bhd":500.63,"bmd":1326.57,"bnb":4.706855,"brl":7147.94,"btc":0.06878001,"cad":1815.07,"chf":1298.33,"clp":1285510,"cny":9451.79,"czk":33259,"dkk":10076.71,"dot":208.407,"eos":1108,"eth":1.0,"eur":1355.02,"gbp":1195.76,"hkd":10413.48,"huf":569452,"idr":20153811,"ils":4728.02,"inr":108128,"jpy":191426,"krw":1892848,"kwd":411.3,"lkr":477052,"ltc":24.895338,"mmk":2786503,"mxn":26695,"myr":6149.3,"ngn":573130,"nok":14196.71,"nzd":2320.57,"php":77667,"pkr":307697,"pln":6577.72,"rub":77438,"sar":4985.41,"sek":14815.17,"sgd":1902.16,"thb":50284,"try":24527,"twd":42037,"uah":48766,"usd":1326.57,"vef":132.83,"vnd":31517393,"xag":70.37,"xau":0.797466,"xdr":1047.35,"xlm":12274,"xrp":3054,"yfi":0.16356957,"zar":23838,"bits":68780,"link":169.959,"sats":6878001},"total_value_locked":null,"mcap_to_tvl_ratio":null,"fdv_to_tvl_ratio":null,"roi":{"times":90.95926735667645,"currency":"btc","percentage":9095.926735667645},"ath":{"aed":17918.33,"ars":488484,"aud":6693.75,"bch":14.621519,"bdt":418411,"bhd":1839.17,"bmd":4878.26,"bnb":7747,"brl":26931,"btc":0.1474984,"cad":6108.18,"chf":4452.99,"clp":3990223,"cny":31155,"czk":107973,"dkk":31448,"dot":258.737,"eos":1566,"eth":1.003981,"eur":4228.93,"gbp":3610.11,"hkd":38002,"huf":1542639,"idr":69531233,"ils":15168.65,"inr":362338,"jpy":553787,"krw":5790811,"kwd":1471.83,"lkr":1064769,"ltc":32.998587,"mmk":8917781,"mxn":102241,"myr":20262,"ngn":2008213,"nok":43275,"nzd":6993.89,"php":244091,"pkr":839917,"pln":19645.56,"rub":403838,"sar":18294.87,"sek":43214,"sgd":6489.66,"thb":161550,"try":70272,"twd":135247,"uah":130195,"usd":4878.26,"vef":283483138,"vnd":110456068,"xag":211.07,"xau":2.69,"xdr":3455.91,"xlm":17299,"xrp":5418,"yfi":0.29805191,"zar":75832,"bits":147498,"link":2891,"sats":14749840},"ath_change_percentage":{"aed":-72.73853,"ars":-59.9522,"aud":-69.40287,"bch":-21.45468,"bdt":-67.8171,"bhd":-72.71079,"bmd":-72.73813,"bnb":-99.93914,"brl":-73.37054,"btc":-53.31246,"cad":-70.18366,"chf":-70.74673,"clp":-67.48306,"cny":-69.58523,"czk":-69.06942,"dkk":-67.84388,"dot":-19.27605,"eos":-29.26984,"eth":-0.39652,"eur":-67.84399,"gbp":-66.66031,"hkd":-72.52899,"huf":-62.89324,"idr":-70.95717,"ils":-68.75187,"inr":-70.06154,"jpy":-65.30831,"krw":-67.22168,"kwd":-71.98491,"lkr":-55.08393,"ltc":-24.5359,"mmk":-68.67479,"mxn":-73.79794,"myr":-69.57447,"ngn":-71.3889,"nok":-67.06842,"nzd":-66.69609,"php":-68.12532,"pkr":-63.27359,"pln":-66.37719,"rub":-80.77615,"sar":-72.68068,"sek":-65.5799,"sgd":-70.60763,"thb":-68.76354,"try":-64.98058,"twd":-68.82483,"uah":-62.44969,"usd":-72.73813,"vef":-99.99995,"vnd":-71.39434,"xag":-66.48968,"xau":-70.2192,"xdr":-69.61758,"xlm":-28.87915,"xrp":-43.60829,"yfi":-44.85668,"zar":-68.47666,"bits":-53.31246,"link":-94.08308,"sats":-53.31246},"ath_date":{"aed":"2021-11-10T14
2022-09-29 23:17:13.666 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  :24:19.604Z","ars":"2021-11-10T14:24:19.604Z","aud":"2021-12-01T08:38:24.623Z","bch":"2022-08-19T08:40:54.729Z","bdt":"2021-11-10T14:24:19.604Z","bhd":"2021-11-10T14:24:19.604Z","bmd":"2021-11-10T14:24:19.604Z","bnb":"2017-10-19T00:00:00.000Z","brl":"2021-11-09T04:03:16.303Z","btc":"2017-06-12T00:00:00.000Z","cad":"2021-12-01T08:38:24.623Z","chf":"2021-11-10T16:19:02.090Z","clp":"2021-12-01T16:19:01.053Z","cny":"2021-11-10T14:24:19.604Z","czk":"2021-12-01T08:38:24.623Z","dkk":"2021-12-01T08:38:24.623Z","dot":"2022-05-11T12:59:46.918Z","eos":"2022-05-12T07:29:33.469Z","eth":"2019-10-15T16:26:30.106Z","eur":"2021-12-01T08:38:24.623Z","gbp":"2021-11-12T02:17:36.494Z","hkd":"2021-11-10T14:24:19.604Z","huf":"2021-12-01T08:38:24.623Z","idr":"2021-11-10T14:24:19.604Z","ils":"2021-11-10T14:24:19.604Z","inr":"2021-11-10T14:24:19.604Z","jpy":"2021-11-10T16:19:02.090Z","krw":"2021-11-08T23:48:45.609Z","kwd":"2021-11-10T14:24:19.604Z","lkr":"2022-04-03T22:43:06.993Z","ltc":"2022-05-12T05:31:14.227Z","mmk":"2021-11-10T14:24:19.604Z","mxn":"2021-12-01T08:38:24.623Z","myr":"2021-11-10T14:24:19.604Z","ngn":"2021-11-09T04:03:16.303Z","nok":"2021-12-01T08:38:24.623Z","nzd":"2021-12-01T08:38:24.623Z","php":"2021-11-10T14:24:19.604Z","pkr":"2021-12-01T08:38:24.623Z","pln":"2021-12-01T08:38:24.623Z","rub":"2022-03-07T16:44:57.811Z","sar":"2021-11-10T14:24:19.604Z","sek":"2021-12-01T08:38:24.623Z","sgd":"2021-11-09T00:00:00.000Z","thb":"2021-12-01T08:38:24.623Z","try":"2021-12-20T16:43:34.305Z","twd":"2021-11-10T14:24:19.604Z","uah":"2021-12-01T08:38:24.623Z","usd":"2021-11-10T14:24:19.604Z","vef":"2021-01-04T06:47:56.457Z","vnd":"2021-11-10T14:24:19.604Z","xag":"2021-12-01T16:19:01.053Z","xau":"2021-12-01T08:38:24.623Z","xdr":"2021-11-10T14:24:19.604Z","xlm":"2022-05-12T05:31:14.227Z","xrp":"2021-12-04T07:02:03.517Z","yfi":"2020-07-19T00:00:00.000Z","zar":"2021-12-01T08:38:24.623Z","bits":"2017-06-12T00:00:00.000Z","link":"2017-11-29T00:00:00.000Z","sats":"2017-06-12T00:00:00.000Z"},"atl":{"aed":1.59,"ars":4.1,"aud":0.596152,"bch":0.20411961,"bdt":33.7,"bhd":0.163288,"bmd":0.432979,"bnb":3.541506,"brl":1.69,"btc":0.00160204,"cad":0.561954,"chf":0.413801,"clp":297.03,"cny":2.81,"czk":10.33,"dkk":2.84,"dot":40.15685,"eos":30.013555,"eth":0.99987722,"eur":0.381455,"gbp":0.280314,"hkd":3.35,"huf":118.42,"idr":5876.48,"ils":1.67,"inr":28.13,"jpy":51.85,"krw":489.4,"kwd":0.130639,"lkr":61.02,"ltc":0.13982924,"mmk":468.1,"mxn":7.16,"myr":1.85,"ngn":86.13,"nok":3.52,"nzd":0.641302,"php":20.05,"pkr":45.2,"pln":1.62,"rub":26.88,"sar":1.62,"sek":3.59,"sgd":0.601615,"thb":15.33,"try":1.26,"twd":14.02,"uah":9.54,"usd":0.432979,"vef":2.74,"vnd":9662.0,"xag":0.02721328,"xau":0.00036799,"xdr":0.306291,"xlm":664.668,"xrp":286.238,"yfi":0.00878891,"zar":5.74,"bits":7718.95,"link":21.519699,"sats":771895},"atl_change_percentage":{"aed":307052.05223,"ars":4766156.7308,"aud":343452.62322,"bch":5526.36744,"bdt":399467.46285,"bhd":307267.48612,"bmd":307052.47035,"bnb":33.13695,"brl":423879.81334,"btc":4198.47019,"cad":323990.01558,"chf":314699.71268,"clp":436728.6508,"cny":336786.7412,"czk":323151.18247,"dkk":355366.4476,"dot":420.11781,"eos":3589.45275,"eth":0.01228,"eur":356391.25289,"gbp":429275.06027,"hkd":311114.13213,"huf":483302.40632,"idr":343538.42758,"ils":284086.72882,"inr":385515.90264,"jpy":370460.77349,"krw":387745.09678,"kwd":315529.36994,"lkr":783642.79322,"ltc":17708.92732,"mmk":596681.32407,"mxn":374218.4574,"myr":333118.24429,"ngn":666966.22945,"nok":405242.11284,"nzd":363104.44575,"php":388006.4432,"pkr":682432.93371,"pln":406632.69119,"rub":288701.00391,"sar":307708.92693,"sek":413771.36058,"sgd":316957.32927,"thb":329084.66661,"try":1959879.65442,"twd":300649.74578,"uah":512118.41436,"usd":307052.47035,"vef":4765.74341,"vnd":326920.30986,"xag":259810.62684,"xau":217425.1203,"xdr":342708.07101,"xlm":1751.0144,"xrp":967.43494,"yfi":1770.03507,"zar":416179.38274,"bits":792.1343,"link":694.79729,"sats":792.1343},"atl_date":{"aed":"2015-10-20T00:00:00.0
2022-09-29 23:17:13.666 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  00Z","ars":"2015-10-20T00:00:00.000Z","aud":"2015-10-20T00:00:00.000Z","bch":"2017-11-12T00:00:00.000Z","bdt":"2015-10-20T00:00:00.000Z","bhd":"2015-10-20T00:00:00.000Z","bmd":"2015-10-20T00:00:00.000Z","bnb":"2021-04-12T21:29:59.636Z","brl":"2015-10-20T00:00:00.000Z","btc":"2015-10-20T00:00:00.000Z","cad":"2015-10-20T00:00:00.000Z","chf":"2015-10-20T00:00:00.000Z","clp":"2015-10-20T00:00:00.000Z","cny":"2015-10-20T00:00:00.000Z","czk":"2015-10-20T00:00:00.000Z","dkk":"2015-10-20T00:00:00.000Z","dot":"2021-03-03T00:43:22.672Z","eos":"2019-04-16T00:00:00.000Z","eth":"2019-10-22T00:00:00.000Z","eur":"2015-10-20T00:00:00.000Z","gbp":"2015-10-20T00:00:00.000Z","hkd":"2015-10-20T00:00:00.000Z","huf":"2015-10-20T00:00:00.000Z","idr":"2015-10-20T00:00:00.000Z","ils":"2015-10-20T00:00:00.000Z","inr":"2015-10-20T00:00:00.000Z","jpy":"2015-10-20T00:00:00.000Z","krw":"2015-10-20T00:00:00.000Z","kwd":"2015-10-20T00:00:00.000Z","lkr":"2015-10-20T00:00:00.000Z","ltc":"2015-10-20T00:00:00.000Z","mmk":"2015-10-20T00:00:00.000Z","mxn":"2015-10-20T00:00:00.000Z","myr":"2015-10-20T00:00:00.000Z","ngn":"2015-10-20T00:00:00.000Z","nok":"2015-10-20T00:00:00.000Z","nzd":"2015-10-20T00:00:00.000Z","php":"2015-10-20T00:00:00.000Z","pkr":"2015-10-20T00:00:00.000Z","pln":"2015-10-20T00:00:00.000Z","rub":"2015-10-20T00:00:00.000Z","sar":"2015-10-20T00:00:00.000Z","sek":"2015-10-20T00:00:00.000Z","sgd":"2015-10-20T00:00:00.000Z","thb":"2015-10-20T00:00:00.000Z","try":"2015-10-20T00:00:00.000Z","twd":"2015-10-20T00:00:00.000Z","uah":"2015-10-20T00:00:00.000Z","usd":"2015-10-20T00:00:00.000Z","vef":"2015-10-20T00:00:00.000Z","vnd":"2015-10-20T00:00:00.000Z","xag":"2015-10-20T00:00:00.000Z","xau":"2015-10-20T00:00:00.000Z","xdr":"2015-10-20T00:00:00.000Z","xlm":"2018-11-20T00:00:00.000Z","xrp":"2018-12-20T00:00:00.000Z","yfi":"2020-09-14T08:23:43.987Z","zar":"2015-10-20T00:00:00.000Z","bits":"2016-12-27T00:00:00.000Z","link":"2020-08-16T08:12:39.777Z","sats":"2016-12-27T00:00:00.000Z"},"market_cap":{"aed":588678312735,"ars":23575211784337,"aud":246789260237,"bch":1379649612,"bdt":16227787246549,"bhd":60484392955,"bmd":160269834085,"bnb":568709409,"brl":862956894645,"btc":8304975,"cad":219432161178,"chf":156945356916,"clp":155437698586916,"cny":1141922567853,"czk":4023894564091,"dkk":1218932223130,"dot":25173742292,"eos":133894394240,"eth":120734247,"eur":163880392907,"gbp":144890020266,"hkd":1258110184072,"huf":68922296809574,"idr":2433449012331294,"ils":571218517010,"inr":13072480016644,"jpy":23143865559628,"krw":228839504564108,"kwd":49691341518,"lkr":57635272268011,"ltc":3004117328,"mmk":336652713753904,"mxn":3226300515881,"myr":742930815899,"ngn":69242979117896,"nok":1716570859312,"nzd":280694984717,"php":9376585681499,"pkr":37174588015915,"pln":795311324963,"rub":9355734576084,"sar":602350451471,"sek":1791957782519,"sgd":229867971155,"thb":6073263810642,"try":2963693744908,"twd":5078390257990,"uah":5891659657593,"usd":160269834085,"vef":16047818487,"vnd":3807789742661065,"xag":8531799540,"xau":96469619,"xdr":126536559946,"xlm":1482763172064,"xrp":368688066064,"yfi":19809044,"zar":2879503039445,"bits":8304975286652,"link":20504464031,"sats":830497528665188},"market_cap_rank":2,"fully_diluted_valuation":{},"total_volume":{"aed":46417249173,"ars":1858723644318,"aud":19437771627,"bch":108845852,"bdt":1279560037881,"bhd":4769190708,"bmd":12637266675,"bnb":44838888,"brl":68093384022,"btc":655219,"cad":17290902216,"chf":12368257179,"clp":12246143270965,"cny":90040525056,"czk":316835193519,"dkk":95993713916,"dot":1985350519,"eos":10555801665,"eth":9529483,"eur":12908285496,"gbp":11391168994,"hkd":99201911532,"huf":5424767543645,"idr":191991153638799,"ils":45040545341,"inr":1030054659457,"jpy":1823579345290,"krw":18031831188747,"kwd":3918159258,"lkr":4544537714604,"ltc":237160338,"mmk":26545046013909,"mxn":254300349155,"myr":58580049670,"ngn":5459804694069,"nok":135242031263,"nzd":22106459055,"php":739874051994,"pkr":2931214005160,"pln":62661254942,"rub":7376991
2022-09-29 23:17:13.666 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  02576,"sar":47492528919,"sek":141133714545,"sgd":18120576685,"thb":479020420734,"try":233652003204,"twd":400456037653,"uah":464557005839,"usd":12637266675,"vef":1265369512,"vnd":300243989727995,"xag":670323970,"xau":7596893,"xdr":9977400059,"xlm":116925205829,"xrp":29089515792,"yfi":1558212,"zar":227086210205,"bits":655218716934,"link":1619080093,"sats":65521871693437},"high_24h":{"aed":4973.63,"ars":198868,"aud":2079.05,"bch":11.773638,"bdt":139087,"bhd":510.86,"bmd":1354.09,"bnb":4.792134,"brl":7279.84,"btc":0.06907022,"cad":1842.82,"chf":1320.82,"clp":1299605,"cny":9750.09,"czk":34349,"dkk":10366.43,"dot":210.325,"eos":1169,"eth":1.0,"eur":1393.81,"gbp":1249.69,"hkd":10628.26,"huf":580411,"idr":20548588,"ils":4780.86,"inr":110198,"jpy":195143,"krw":1933290,"kwd":420.06,"lkr":493782,"ltc":25.245497,"mmk":2848763,"mxn":27293,"myr":6258.42,"ngn":585074,"nok":14474.4,"nzd":2367.99,"php":79524,"pkr":316905,"pln":6709.72,"rub":79146,"sar":5100.48,"sek":15319.23,"sgd":1941.82,"thb":51237,"try":25063,"twd":42950,"uah":49857,"usd":1354.09,"vef":135.58,"vnd":32144471,"xag":71.82,"xau":0.815864,"xdr":1054.28,"xlm":12375,"xrp":3563,"yfi":0.16431094,"zar":24207,"bits":69070,"link":174.399,"sats":6907022},"low_24h":{"aed":4756.89,"ars":190525,"aud":2001.24,"bch":11.270123,"bdt":131131,"bhd":488.57,"bmd":1295.08,"bnb":4.641298,"brl":7006.76,"btc":0.06765578,"cad":1778.3,"chf":1270.07,"clp":1249671,"cny":9236.0,"czk":32695,"dkk":9889.95,"dot":205.119,"eos":1094,"eth":1.0,"eur":1329.98,"gbp":1178.27,"hkd":10166.37,"huf":558679,"idr":19680351,"ils":4604.82,"inr":105840,"jpy":187224,"krw":1854710,"kwd":401.75,"lkr":465729,"ltc":24.611387,"mmk":2720365,"mxn":26217,"myr":6003.35,"ngn":559527,"nok":13957.63,"nzd":2280.63,"php":75832,"pkr":296591,"pln":6462.57,"rub":75699,"sar":4867.34,"sek":14569.39,"sgd":1861.3,"thb":49310,"try":24003,"twd":41080,"uah":47608,"usd":1295.08,"vef":129.68,"vnd":30769334,"xag":69.38,"xau":0.783361,"xdr":989.33,"xlm":12068,"xrp":2854,"yfi":0.16060881,"zar":23366,"bits":67656,"link":164.068,"sats":6765578},"price_change_24h":-11.83334803686,"price_change_percentage_24h":-0.88414,"price_change_percentage_7d":5.8903,"price_change_percentage_14d":-18.88268,"price_change_percentage_30d":-14.69997,"price_change_percentage_60d":-21.81881,"price_change_percentage_200d":-48.57188,"price_change_percentage_1y":-52.60542,"market_cap_change_24h":-1883074944.348,"market_cap_change_percentage_24h":-1.1613,"price_change_24h_in_currency":{"aed":-43.477803841336936,"ars":-1452.7839606594061,"aud":-14.598266301851709,"bch":-0.3191656074238942,"bdt":-3157.0139196971722,"bhd":-4.278423030910233,"bmd":-11.833348036858979,"bnb":-0.062036729769069154,"brl":-21.468185148735756,"btc":0.00050147,"cad":-10.550924807533647,"chf":-7.746935187440158,"clp":1180.78,"cny":-185.36181450669392,"czk":-763.2339891952579,"dkk":-190.9483373677158,"dot":0.90134339,"eos":-51.45455859366416,"eth":0.0,"eur":-25.629058524687025,"gbp":-38.60026984729461,"hkd":-92.72121360557139,"huf":-1025.5597266132245,"idr":-199184.8722877875,"ils":2.54,"inr":-827.9574713049369,"jpy":-1528.375593691453,"krw":-14148.954239486251,"kwd":-4.00618272465573,"lkr":-11010.334757892007,"ltc":-0.003651073281083939,"mmk":-29260.831142453942,"mxn":-460.7003340759329,"myr":-22.062681662899195,"ngn":-4162.215594212059,"nok":-138.02840611977444,"nzd":-24.43831011478096,"php":-1059.5340164771624,"pkr":-9169.065696637379,"pln":-64.32445151813408,"rub":-791.2949942103587,"sar":-55.80486805068995,"sek":-202.9515186032604,"sgd":-20.609624501224516,"thb":-494.21111327944527,"try":-254.08855442590357,"twd":-410.44598771923484,"uah":-513.9471064935788,"usd":-11.833348036858979,"vef":-1.184873138930783,"vnd":-254720.7004112266,"xag":-0.9398096623575327,"xau":-0.009429049395634716,"xdr":43.89,"xlm":124.277,"xrp":12.002075,"yfi":0.00175685,"zar":-43.57745128017268,"bits":501.47,"link":5.890609,"sats":50147},"price_change_percentage_1h_in_currency":{"aed":0.15089,"ars":0.13778,"aud":-0.0759,"bch":0.1328
2022-09-29 23:17:13.666 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  1,"bdt":0.15089,"bhd":0.15381,"bmd":0.15089,"bnb":0.31886,"brl":0.02651,"btc":-0.19168,"cad":0.04939,"chf":0.10046,"clp":0.64943,"cny":0.14245,"czk":-0.16392,"dkk":-0.00409,"dot":-0.12635,"eos":0.3329,"eth":0.0,"eur":-0.00672,"gbp":-0.31459,"hkd":0.15242,"huf":-0.10193,"idr":0.32487,"ils":0.15089,"inr":-0.0536,"jpy":-0.00722,"krw":-0.1415,"kwd":0.16155,"lkr":0.15089,"ltc":0.1908,"mmk":0.15089,"mxn":-0.13175,"myr":0.15089,"ngn":0.15089,"nok":-0.01224,"nzd":-0.13558,"php":0.16542,"pkr":0.15089,"pln":-0.09693,"rub":0.09498,"sar":0.14439,"sek":-0.06877,"sgd":0.07314,"thb":-0.07745,"try":-0.02998,"twd":0.09719,"uah":0.15089,"usd":0.15089,"vef":0.15089,"vnd":0.15089,"xag":-0.33241,"xau":-0.1961,"xdr":0.15089,"xlm":0.14991,"xrp":0.1204,"yfi":-0.3891,"zar":-0.0947,"bits":-0.19168,"link":0.64488,"sats":-0.19168},"price_change_percentage_24h_in_currency":{"aed":-0.88441,"ars":-0.73908,"aud":-0.71037,"bch":-2.71746,"bdt":-2.29641,"bhd":-0.84736,"bmd":-0.88414,"bnb":-1.30086,"brl":-0.29944,"btc":0.73445,"cad":-0.57794,"chf":-0.59315,"clp":0.09194,"cny":-1.92341,"czk":-2.24334,"dkk":-1.85971,"dot":0.43437,"eos":-4.43756,"eth":0.0,"eur":-1.85631,"gbp":-3.12715,"hkd":-0.88254,"huf":-0.17977,"idr":-0.97865,"ils":0.05376,"inr":-0.7599,"jpy":-0.79209,"krw":-0.74195,"kwd":-0.96463,"lkr":-2.25593,"ltc":-0.01466,"mmk":-1.03918,"mxn":-1.69654,"myr":-0.3575,"ngn":-0.72099,"nok":-0.96289,"nzd":-1.04214,"php":-1.34585,"pkr":-2.89367,"pln":-0.96844,"rub":-1.0115,"sar":-1.10697,"sek":-1.35138,"sgd":-1.07187,"thb":-0.97327,"try":-1.02533,"twd":-0.96695,"uah":-1.04292,"usd":-0.88414,"vef":-0.88414,"vnd":-0.80171,"xag":-1.31801,"xau":-1.16856,"xdr":4.37436,"xlm":1.02288,"xrp":0.3946,"yfi":1.08573,"zar":-0.18247,"bits":0.73445,"link":3.59034,"sats":0.73445},"price_change_percentage_7d_in_currency":{"aed":5.89117,"ars":7.4885,"aud":7.67206,"bch":-0.43697,"bdt":3.46304,"bhd":5.99253,"bmd":5.8903,"bnb":-0.5958,"brl":10.32091,"btc":1.76262,"cad":7.4933,"chf":7.13271,"clp":9.41023,"cny":7.03198,"czk":5.87044,"dkk":6.32678,"dot":2.34496,"eos":5.28387,"eth":0.0,"eur":6.34653,"gbp":7.42929,"hkd":5.89273,"huf":9.75554,"idr":6.82874,"ils":9.03884,"inr":7.81945,"jpy":5.88937,"krw":8.09616,"kwd":6.14982,"lkr":5.99713,"ltc":1.7576,"mmk":5.83582,"mxn":6.35001,"myr":7.82087,"ngn":6.42484,"nok":9.29245,"nzd":8.0103,"php":6.83831,"pkr":2.37046,"pln":7.98091,"rub":0.34636,"sar":5.78002,"sek":6.57453,"sgd":7.02044,"thb":7.81124,"try":6.74316,"twd":6.67636,"uah":5.32161,"usd":5.8903,"vef":5.8903,"vnd":6.16706,"xag":9.30238,"xau":6.07558,"xdr":12.62584,"xlm":8.847,"xrp":-2.86754,"yfi":5.04499,"zar":7.2203,"bits":1.76262,"link":-8.96495,"sats":1.76262},"price_change_percentage_14d_in_currency":{"aed":-18.88187,"ars":-16.48117,"aud":-15.74679,"bch":-15.978,"bdt":-15.23543,"bhd":-18.79188,"bmd":-18.88268,"bnb":-19.53223,"brl":-15.36425,"btc":-14.82472,"cad":-15.66593,"chf":-17.47531,"clp":-14.59967,"cny":-16.9835,"czk":-17.08519,"dkk":-17.29433,"dot":-8.14936,"eos":0.04398,"eth":0.0,"eur":-17.29167,"gbp":-15.59388,"hkd":-18.87132,"huf":-14.32045,"idr":-17.41521,"ils":-15.60021,"inr":-16.78442,"jpy":-18.08262,"krw":-16.82335,"kwd":-18.55169,"lkr":-20.08369,"ltc":-8.56072,"mmk":-18.86633,"mxn":-18.24812,"myr":-16.93852,"ngn":-18.00487,"nok":-13.97995,"nzd":-14.63813,"php":-16.81976,"pkr":-19.88676,"pln":-14.89079,"rub":-20.71638,"sar":-18.92107,"sek":-15.25789,"sgd":-17.24537,"thb":-16.10415,"try":-17.82667,"twd":-17.29373,"uah":-19.26121,"usd":-18.88268,"vef":-18.88268,"vnd":-18.34306,"xag":-15.4673,"xau":-17.20657,"xdr":-10.02264,"xlm":-20.98016,"xrp":-36.06806,"yfi":-7.84499,"zar":-16.64634,"bits":-14.82472,"link":-22.57874,"sats":-14.82472},"price_change_percentage_30d_in_currency":{"aed":-14.69886,"ars":-9.29081,"aud":-9.38001,"bch":-12.08887,"bdt":-9.15403,"bhd":-14.61966,"bmd":-14.69997,"bnb":-13.31736,"brl":-8.60019,"btc":-10.15348,"cad":-10.24677,"chf":-13.69782,"clp":-6.33535,"cny":-12.02046,"czk":-12.96282,"dkk":-12.80149,"dot":-2.33077,"eos":6.55753,"eth":0.0,"eur":-12.7
2022-09-29 23:17:13.666 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  8778,"gbp":-9.88107,"hkd":-14.67068,"huf":-10.11393,"idr":-12.95091,"ils":-8.45443,"inr":-13.05607,"jpy":-11.18658,"krw":-9.57923,"kwd":-14.17068,"lkr":-14.59431,"ltc":-10.61162,"mmk":-14.72084,"mxn":-14.16701,"myr":-11.83762,"ngn":-13.54158,"nok":-6.25046,"nzd":-8.09767,"php":-11.2167,"pkr":-10.96045,"pln":-10.58487,"rub":-19.36226,"sar":-14.64571,"sek":-10.60521,"sgd":-12.34492,"thb":-11.02559,"try":-13.27485,"twd":-11.03479,"uah":-15.13634,"usd":-14.69997,"vef":-14.69997,"vnd":-13.52452,"xag":-14.96479,"xau":-10.79741,"xdr":-8.84534,"xlm":-16.30885,"xrp":-34.37399,"yfi":-4.41311,"zar":-9.02614,"bits":-10.15348,"link":-26.57409,"sats":-10.15348},"price_change_percentage_60d_in_currency":{"aed":-21.81998,"ars":-12.35827,"aud":-15.91599,"bch":-2.60469,"bdt":-16.51183,"bhd":-21.70076,"bmd":-21.81881,"bnb":-20.28887,"brl":-18.56176,"btc":-4.15615,"cad":-16.39774,"chf":-19.64033,"clp":-16.0519,"cny":-17.40444,"czk":-18.5794,"dkk":-18.41482,"dot":0.91078,"eos":-14.62723,"eth":0.0,"eur":-18.33505,"gbp":-14.20015,"hkd":-21.81901,"huf":-15.12995,"idr":-19.82906,"ils":-18.26299,"inr":-19.53414,"jpy":-15.30589,"krw":-14.40444,"kwd":-21.01301,"lkr":-21.58136,"ltc":-11.41509,"mmk":-11.43538,"mxn":-22.75524,"myr":-18.56895,"ngn":-18.77894,"nok":-13.37836,"nzd":-14.11993,"php":-17.3113,"pkr":-24.15905,"pln":-16.34399,"rub":-26.68565,"sar":-21.77689,"sek":-13.93475,"sgd":-18.79816,"thb":-18.2803,"try":-19.31242,"twd":-17.45152,"uah":-22.25601,"usd":-21.81881,"vef":-21.81881,"vnd":-20.41658,"xag":-15.60877,"xau":-16.9929,"xdr":-16.14998,"xlm":-13.64825,"xrp":-30.3295,"yfi":13.03559,"zar":-15.40529,"bits":-4.15615,"link":-21.74235,"sats":-4.15615},"price_change_percentage_200d_in_currency":{"aed":-48.57265,"ars":-30.47244,"aud":-42.3257,"bch":28.94234,"bdt":-39.38461,"bhd":-48.49122,"bmd":-48.57188,"bnb":-31.85766,"brl":-45.39702,"btc":3.89145,"cad":-44.78995,"chf":-46.14586,"clp":-38.16515,"cny":-42.20238,"czk":-43.97995,"dkk":-42.69987,"dot":46.79011,"eos":-15.29351,"eth":0.0,"eur":-42.66759,"gbp":-39.55733,"hkd":-48.43756,"huf":-36.98034,"idr":-45.46524,"ils":-43.61622,"inr":-45.38935,"jpy":-36.72699,"krw":-40.68278,"kwd":-47.48637,"lkr":-27.34246,"ltc":2.15097,"mmk":-39.13792,"mxn":-50.53518,"myr":-43.16485,"ngn":-46.56838,"nok":-38.89942,"nzd":-38.7661,"php":-42.52408,"pkr":-33.24705,"pln":-41.86557,"rub":-77.59619,"sar":-48.48128,"sek":-41.07731,"sgd":-45.84701,"thb":-41.4981,"try":-35.59816,"twd":-42.64215,"uah":-35.5807,"usd":-48.57188,"vef":-48.57188,"vnd":-46.59246,"xag":-29.5651,"xau":-38.5392,"xdr":-43.28904,"xlm":-11.69434,"xrp":-6.98716,"yfi":16.73474,"zar":-38.53348,"bits":3.89145,"link":-13.10095,"sats":3.89145},"price_change_percentage_1y_in_currency":{"aed":-52.60742,"ars":-29.28061,"aud":-47.20669,"bch":93.70257,"bdt":-43.97175,"bhd":-52.55753,"bmd":-52.60542,"bnb":-44.14495,"brl":-52.96593,"btc":1.3196,"cad":-48.8618,"chf":-50.08062,"clp":-42.71186,"cny":-47.72657,"czk":-45.54419,"dkk":-43.42289,"dot":95.9204,"eos":46.20441,"eth":0.0,"eur":-43.42738,"gbp":-42.15886,"hkd":-52.19988,"huf":-33.90685,"idr":-49.71313,"ils":-47.37833,"inr":-47.93707,"jpy":-38.68681,"krw":-43.06689,"kwd":-51.2237,"lkr":-14.67498,"ltc":25.14698,"mmk":-46.80559,"mxn":-53.10786,"myr":-47.47219,"ngn":-50.15515,"nok":-41.43798,"nzd":-42.31604,"php":-45.74827,"pkr":-35.40287,"pln":-40.76225,"rub":-62.03667,"sar":-52.5165,"sek":-39.39164,"sgd":-49.92534,"thb":-46.90977,"try":-1.25908,"twd":-45.88485,"uah":-34.49027,"usd":-52.60542,"vef":-52.60542,"vnd":-50.5109,"xag":-43.4973,"xau":-50.58407,"xdr":-47.01463,"xlm":12.88397,"xrp":-1.9942,"yfi":64.92974,"zar":-43.63153,"bits":1.3196,"link":34.20475,"sats":1.3196},"market_cap_change_24h_in_currency":{"aed":-6918240538.055542,"ars":-233103774284.54688,"aud":-2038918948.880371,"bch":-41826894.277243376,"bdt":-427990250037.25195,"bhd":-687954129.0587006,"bmd":-1883074944.3485718,"bnb":-9290299.869598627,"brl":-5112488550.455811,"btc":32229,"cad":-1693490723.7938843,"chf":-1156485986.4953308,"clp":-133423864514
2022-09-29 23:17:13.667 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  .71875,"cny":-25659453610.16577,"czk":-96158382254.00098,"dkk":-24021389201.534912,"dot":117658157,"eos":-6523338939.160385,"eth":14937,"eur":-3266339261.455475,"gbp":-4420702673.499939,"hkd":-14765828868.483643,"huf":-137978534204.25,"idr":-31758044868821.0,"ils":-1293948133.871704,"inr":-154179700498.3086,"jpy":-223233446412.83203,"krw":-2488678813820.9688,"kwd":-610761003.2442169,"lkr":-1495564826794.6875,"ltc":-6886112.174203873,"mmk":-4489086057831.3125,"mxn":-64191930869.57373,"myr":-4756247633.319702,"ngn":-698435133545.8906,"nok":-19126364707.588867,"nzd":-3293944804.0578613,"php":-165059345250.20312,"pkr":-1215113196680.4375,"pln":-8734414395.597778,"rub":-122103280961.75586,"sar":-8398642538.919312,"sek":-26638332559.09326,"sgd":-3065006971.0610046,"thb":-78804423529.52148,"try":-38632389078.48584,"twd":-69526628670.31934,"uah":-78787751444.24219,"usd":-1883074944.3485718,"vef":-188552294.17759514,"vnd":-41537962794006.5,"xag":-88792316.65958023,"xau":-1219401.5122876912,"xdr":4963227166,"xlm":12676465667,"xrp":758425865,"yfi":231218,"zar":-13122273455.515625,"bits":32228768819,"link":747262057,"sats":3222876881867},"market_cap_change_percentage_24h_in_currency":{"aed":-1.16156,"ars":-0.97909,"aud":-0.81941,"bch":-2.9425,"bdt":-2.56962,"bhd":-1.12462,"bmd":-1.1613,"bnb":-1.60732,"brl":-0.58895,"btc":0.38958,"cad":-0.76585,"chf":-0.73148,"clp":-0.08576,"cny":-2.19766,"czk":-2.33391,"dkk":-1.93261,"dot":0.46958,"eos":-4.64567,"eth":0.01237,"eur":-1.95417,"gbp":-2.96074,"hkd":-1.16004,"huf":-0.19979,"idr":-1.28825,"ils":-0.22601,"inr":-1.16567,"jpy":-0.95533,"krw":-1.07582,"kwd":-1.21419,"lkr":-2.52925,"ltc":-0.2287,"mmk":-1.3159,"mxn":-1.95083,"myr":-0.63613,"ngn":-0.9986,"nok":-1.10194,"nzd":-1.15988,"php":-1.72988,"pkr":-3.16521,"pln":-1.08631,"rub":-1.2883,"sar":-1.37514,"sek":-1.46477,"sgd":-1.31583,"thb":-1.28094,"try":-1.28675,"twd":-1.35058,"uah":-1.31963,"usd":-1.1613,"vef":-1.1613,"vnd":-1.0791,"xag":-1.03,"xau":-1.24825,"xdr":4.0825,"xlm":0.86229,"xrp":0.20613,"yfi":1.18102,"zar":-0.45365,"bits":0.38958,"link":3.78223,"sats":0.38958},"total_supply":120732090.871192,"max_supply":null,"circulating_supply":120734246.871192,"last_updated":"2022-09-29T16:15:38.516Z"},"public_interest_stats":{"alexa_rank":8793,"bing_matches":null},"status_updates":[],"last_updated":"2022-09-29T16:15:38.516Z"}
2022-09-29 23:17:13.667 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- END HTTP (30345-byte body)
2022-09-29 23:17:13.702 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:13.703 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:13.726 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:13.727 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:14.170 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- 200 https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false (466ms)
2022-09-29 23:17:14.170 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  date: Thu, 29 Sep 2022 16:17:13 GMT
2022-09-29 23:17:14.170 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  content-type: application/json; charset=utf-8
2022-09-29 23:17:14.171 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-frame-options: SAMEORIGIN
2022-09-29 23:17:14.171 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-xss-protection: 1; mode=block
2022-09-29 23:17:14.171 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-content-type-options: nosniff
2022-09-29 23:17:14.171 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-download-options: noopen
2022-09-29 23:17:14.171 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-permitted-cross-domain-policies: none
2022-09-29 23:17:14.172 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  referrer-policy: strict-origin-when-cross-origin
2022-09-29 23:17:14.172 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cache-control: public, max-age=300
2022-09-29 23:17:14.172 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-allow-origin: *
2022-09-29 23:17:14.172 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-allow-methods: POST, PUT, DELETE, GET, OPTIONS
2022-09-29 23:17:14.172 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-request-method: *
2022-09-29 23:17:14.172 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Authorization
2022-09-29 23:17:14.173 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  access-control-expose-headers: link, per-page, total
2022-09-29 23:17:14.173 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  vary: Accept-Encoding, Origin
2022-09-29 23:17:14.173 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  etag: W/"ec6e3f0e56f6a8daca4ec5b67e706c09"
2022-09-29 23:17:14.173 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-request-id: 9dff4cb7-7c9b-4eeb-a8a2-057c6000a0dd
2022-09-29 23:17:14.173 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  x-runtime: 0.069083
2022-09-29 23:17:14.173 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  alternate-protocol: 443:npn-spdy/2
2022-09-29 23:17:14.174 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-cache-status: HIT
2022-09-29 23:17:14.174 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  age: 8
2022-09-29 23:17:14.174 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  expires: Thu, 29 Sep 2022 16:22:13 GMT
2022-09-29 23:17:14.174 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  server: cloudflare
2022-09-29 23:17:14.174 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-ray: 7526071c6c77e68e-HKG
2022-09-29 23:17:14.203 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  {"id":"ethereum","symbol":"eth","name":"Ethereum","asset_platform_id":null,"platforms":{"":""},"detail_platforms":{"":{"decimal_place":null,"contract_address":""}},"block_time_in_minutes":0,"hashing_algorithm":"Ethash","categories":["Smart Contract Platform","Ethereum Ecosystem"],"public_notice":null,"additional_notices":[],"description":{"en":"Ethereum is a \u003ca href=\"https://www.coingecko.com/en?category_id=29\u0026view=market\"\u003esmart contract platform\u003c/a\u003e that enables developers to build tokens and decentralized applications (dapps). ETH is the native currency for the Ethereum platform and also works as the transaction fees to miners on the Ethereum network.\r\n\r\nEthereum is the pioneer for blockchain based smart contracts. Smart contract is essentially a computer code that runs exactly as programmed without any possibility of downtime, censorship, fraud or third-party interference. It can facilitate the exchange of money, content, property, shares, or anything of value. When running on the blockchain a smart contract becomes like a self-operating computer program that automatically executes when specific conditions are met.\r\n\r\nEthereum allows programmers to run complete-turing smart contracts that is capable of any customizations. Rather than giving a set of limited operations, Ethereum allows developers to have complete control over customization of their smart contract, giving developers the power to build unique and innovative applications.\r\n\r\nEthereum being the first blockchain based smart contract platform, they have gained much popularity, resulting in new competitors fighting for market share. The competitors includes: \u003ca href=\"https://www.coingecko.com/en/coins/ethereum_classic\"\u003eEthereum Classic\u003c/a\u003e which is the oldchain of Ethereum, \u003ca href=\"https://www.coingecko.com/en/coins/qtum\"\u003eQtum\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/eos\"\u003eEOS\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/neo\"\u003eNeo\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/icon\"\u003eIcon\u003c/a\u003e, \u003ca href=\"https://www.coingecko.com/en/coins/tron\"\u003eTron\u003c/a\u003e and \u003ca href=\"https://www.coingecko.com/en/coins/cardano\"\u003eCardano\u003c/a\u003e.\r\n\r\nEthereum wallets are fairly simple to set up with multiple popular choices such as myetherwallet, \u003ca href=\"https://www.coingecko.com/buzz/complete-beginners-guide-to-metamask?locale=en\"\u003emetamask\u003c/a\u003e, and \u003ca href=\"https://www.coingecko.com/buzz/trezor-model-t-wallet-review\"\u003eTrezor\u003c/a\u003e. Read here for more guide on using ethereum wallet: \u003ca href=\"https://www.coingecko.com/buzz/how-to-use-an-ethereum-wallet\"\u003eHow to Use an Ethereum Wallet\u003c/a\u003e"},"links":{"homepage":["https://www.ethereum.org/","",""],"blockchain_site":["https://etherscan.io/","https://ethplorer.io/","https://blockchair.com/ethereum","https://eth.tokenview.io/","","","","","",""],"official_forum_url":["https://forum.ethereum.org/","",""],"chat_url":["","https://discord.com/invite/CetY6Y4",""],"announcement_url":["",""],"twitter_screen_name":"ethereum","facebook_username":"ethereumproject","bitcointalk_thread_identifier":428589,"telegram_channel_identifier":"","subreddit_url":"https://www.reddit.com/r/ethereum","repos_url":{"github":["https://github.com/ethereum/go-ethereum","https://github.com/ethereum/py-evm","https://github.com/ethereum/aleth","https://github.com/ethereum/web3.py","https://github.com/ethereum/solidity","https://github.com/ethereum/sharding","https://github.com/ethereum/casper","https://github.com/paritytech/parity"],"bitbucket":[]}},"image":{"thumb":"https://assets.coingecko.com/coins/images/279/thumb/ethereum.png?1595348880","small":"https://assets.coingecko.com/coins/images/279/small/ethereum.png?1595348880","large":"https://assets.coingecko.com/coins/images/279/large/ethereum.png?1595348880"},"c
2022-09-29 23:17:14.203 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  ountry_origin":"","genesis_date":"2015-07-30","sentiment_votes_up_percentage":69.53,"sentiment_votes_down_percentage":30.47,"ico_data":{"ico_start_date":"2014-07-20T00:00:00.000Z","ico_end_date":"2014-09-01T00:00:00.000Z","short_desc":"A decentralized platform for applications","description":null,"links":{},"softcap_currency":"","hardcap_currency":"","total_raised_currency":"","softcap_amount":null,"hardcap_amount":null,"total_raised":null,"quote_pre_sale_currency":"","base_pre_sale_amount":null,"quote_pre_sale_amount":null,"quote_public_sale_currency":"BTC","base_public_sale_amount":1.0,"quote_public_sale_amount":0.00074794,"accepting_currencies":"","country_origin":"","pre_sale_start_date":null,"pre_sale_end_date":null,"whitelist_url":"","whitelist_start_date":null,"whitelist_end_date":null,"bounty_detail_url":"","amount_for_sale":null,"kyc_required":true,"whitelist_available":null,"pre_sale_available":null,"pre_sale_ended":false},"market_cap_rank":2,"coingecko_rank":2,"coingecko_score":78.783,"developer_score":97.494,"community_score":72.746,"liquidity_score":95.183,"public_interest_score":0.236,"market_data":{"current_price":{"aed":4872.54,"ars":195115,"aud":2040.43,"bch":11.425832,"bdt":134319,"bhd":500.63,"bmd":1326.57,"bnb":4.706855,"brl":7147.94,"btc":0.06878001,"cad":1815.07,"chf":1298.33,"clp":1285510,"cny":9451.79,"czk":33259,"dkk":10076.71,"dot":208.407,"eos":1108,"eth":1.0,"eur":1355.02,"gbp":1195.76,"hkd":10413.48,"huf":569452,"idr":20153811,"ils":4728.02,"inr":108128,"jpy":191426,"krw":1892848,"kwd":411.3,"lkr":477052,"ltc":24.895338,"mmk":2786503,"mxn":26695,"myr":6149.3,"ngn":573130,"nok":14196.71,"nzd":2320.57,"php":77667,"pkr":307697,"pln":6577.72,"rub":77438,"sar":4985.41,"sek":14815.17,"sgd":1902.16,"thb":50284,"try":24527,"twd":42037,"uah":48766,"usd":1326.57,"vef":132.83,"vnd":31517393,"xag":70.37,"xau":0.797466,"xdr":1047.35,"xlm":12274,"xrp":3054,"yfi":0.16356957,"zar":23838,"bits":68780,"link":169.959,"sats":6878001},"total_value_locked":null,"mcap_to_tvl_ratio":null,"fdv_to_tvl_ratio":null,"roi":{"times":90.95926735667645,"currency":"btc","percentage":9095.926735667645},"ath":{"aed":17918.33,"ars":488484,"aud":6693.75,"bch":14.621519,"bdt":418411,"bhd":1839.17,"bmd":4878.26,"bnb":7747,"brl":26931,"btc":0.1474984,"cad":6108.18,"chf":4452.99,"clp":3990223,"cny":31155,"czk":107973,"dkk":31448,"dot":258.737,"eos":1566,"eth":1.003981,"eur":4228.93,"gbp":3610.11,"hkd":38002,"huf":1542639,"idr":69531233,"ils":15168.65,"inr":362338,"jpy":553787,"krw":5790811,"kwd":1471.83,"lkr":1064769,"ltc":32.998587,"mmk":8917781,"mxn":102241,"myr":20262,"ngn":2008213,"nok":43275,"nzd":6993.89,"php":244091,"pkr":839917,"pln":19645.56,"rub":403838,"sar":18294.87,"sek":43214,"sgd":6489.66,"thb":161550,"try":70272,"twd":135247,"uah":130195,"usd":4878.26,"vef":283483138,"vnd":110456068,"xag":211.07,"xau":2.69,"xdr":3455.91,"xlm":17299,"xrp":5418,"yfi":0.29805191,"zar":75832,"bits":147498,"link":2891,"sats":14749840},"ath_change_percentage":{"aed":-72.73853,"ars":-59.9522,"aud":-69.40287,"bch":-21.45468,"bdt":-67.8171,"bhd":-72.71079,"bmd":-72.73813,"bnb":-99.93914,"brl":-73.37054,"btc":-53.31246,"cad":-70.18366,"chf":-70.74673,"clp":-67.48306,"cny":-69.58523,"czk":-69.06942,"dkk":-67.84388,"dot":-19.27605,"eos":-29.26984,"eth":-0.39652,"eur":-67.84399,"gbp":-66.66031,"hkd":-72.52899,"huf":-62.89324,"idr":-70.95717,"ils":-68.75187,"inr":-70.06154,"jpy":-65.30831,"krw":-67.22168,"kwd":-71.98491,"lkr":-55.08393,"ltc":-24.5359,"mmk":-68.67479,"mxn":-73.79794,"myr":-69.57447,"ngn":-71.3889,"nok":-67.06842,"nzd":-66.69609,"php":-68.12532,"pkr":-63.27359,"pln":-66.37719,"rub":-80.77615,"sar":-72.68068,"sek":-65.5799,"sgd":-70.60763,"thb":-68.76354,"try":-64.98058,"twd":-68.82483,"uah":-62.44969,"usd":-72.73813,"vef":-99.99995,"vnd":-71.39434,"xag":-66.48968,"xau":-70.2192,"xdr":-69.61758,"xlm":-28.87915,"xrp":-43.60829,"yfi":-44.85668,"zar":-68.47666,"bits":-53.31246,"link":-94.08308,"sats":-53.31246},"ath_date":{"aed":"2021-11-10T14
2022-09-29 23:17:14.203 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  :24:19.604Z","ars":"2021-11-10T14:24:19.604Z","aud":"2021-12-01T08:38:24.623Z","bch":"2022-08-19T08:40:54.729Z","bdt":"2021-11-10T14:24:19.604Z","bhd":"2021-11-10T14:24:19.604Z","bmd":"2021-11-10T14:24:19.604Z","bnb":"2017-10-19T00:00:00.000Z","brl":"2021-11-09T04:03:16.303Z","btc":"2017-06-12T00:00:00.000Z","cad":"2021-12-01T08:38:24.623Z","chf":"2021-11-10T16:19:02.090Z","clp":"2021-12-01T16:19:01.053Z","cny":"2021-11-10T14:24:19.604Z","czk":"2021-12-01T08:38:24.623Z","dkk":"2021-12-01T08:38:24.623Z","dot":"2022-05-11T12:59:46.918Z","eos":"2022-05-12T07:29:33.469Z","eth":"2019-10-15T16:26:30.106Z","eur":"2021-12-01T08:38:24.623Z","gbp":"2021-11-12T02:17:36.494Z","hkd":"2021-11-10T14:24:19.604Z","huf":"2021-12-01T08:38:24.623Z","idr":"2021-11-10T14:24:19.604Z","ils":"2021-11-10T14:24:19.604Z","inr":"2021-11-10T14:24:19.604Z","jpy":"2021-11-10T16:19:02.090Z","krw":"2021-11-08T23:48:45.609Z","kwd":"2021-11-10T14:24:19.604Z","lkr":"2022-04-03T22:43:06.993Z","ltc":"2022-05-12T05:31:14.227Z","mmk":"2021-11-10T14:24:19.604Z","mxn":"2021-12-01T08:38:24.623Z","myr":"2021-11-10T14:24:19.604Z","ngn":"2021-11-09T04:03:16.303Z","nok":"2021-12-01T08:38:24.623Z","nzd":"2021-12-01T08:38:24.623Z","php":"2021-11-10T14:24:19.604Z","pkr":"2021-12-01T08:38:24.623Z","pln":"2021-12-01T08:38:24.623Z","rub":"2022-03-07T16:44:57.811Z","sar":"2021-11-10T14:24:19.604Z","sek":"2021-12-01T08:38:24.623Z","sgd":"2021-11-09T00:00:00.000Z","thb":"2021-12-01T08:38:24.623Z","try":"2021-12-20T16:43:34.305Z","twd":"2021-11-10T14:24:19.604Z","uah":"2021-12-01T08:38:24.623Z","usd":"2021-11-10T14:24:19.604Z","vef":"2021-01-04T06:47:56.457Z","vnd":"2021-11-10T14:24:19.604Z","xag":"2021-12-01T16:19:01.053Z","xau":"2021-12-01T08:38:24.623Z","xdr":"2021-11-10T14:24:19.604Z","xlm":"2022-05-12T05:31:14.227Z","xrp":"2021-12-04T07:02:03.517Z","yfi":"2020-07-19T00:00:00.000Z","zar":"2021-12-01T08:38:24.623Z","bits":"2017-06-12T00:00:00.000Z","link":"2017-11-29T00:00:00.000Z","sats":"2017-06-12T00:00:00.000Z"},"atl":{"aed":1.59,"ars":4.1,"aud":0.596152,"bch":0.20411961,"bdt":33.7,"bhd":0.163288,"bmd":0.432979,"bnb":3.541506,"brl":1.69,"btc":0.00160204,"cad":0.561954,"chf":0.413801,"clp":297.03,"cny":2.81,"czk":10.33,"dkk":2.84,"dot":40.15685,"eos":30.013555,"eth":0.99987722,"eur":0.381455,"gbp":0.280314,"hkd":3.35,"huf":118.42,"idr":5876.48,"ils":1.67,"inr":28.13,"jpy":51.85,"krw":489.4,"kwd":0.130639,"lkr":61.02,"ltc":0.13982924,"mmk":468.1,"mxn":7.16,"myr":1.85,"ngn":86.13,"nok":3.52,"nzd":0.641302,"php":20.05,"pkr":45.2,"pln":1.62,"rub":26.88,"sar":1.62,"sek":3.59,"sgd":0.601615,"thb":15.33,"try":1.26,"twd":14.02,"uah":9.54,"usd":0.432979,"vef":2.74,"vnd":9662.0,"xag":0.02721328,"xau":0.00036799,"xdr":0.306291,"xlm":664.668,"xrp":286.238,"yfi":0.00878891,"zar":5.74,"bits":7718.95,"link":21.519699,"sats":771895},"atl_change_percentage":{"aed":307052.05223,"ars":4766156.7308,"aud":343452.62322,"bch":5526.36744,"bdt":399467.46285,"bhd":307267.48612,"bmd":307052.47035,"bnb":33.13695,"brl":423879.81334,"btc":4198.47019,"cad":323990.01558,"chf":314699.71268,"clp":436728.6508,"cny":336786.7412,"czk":323151.18247,"dkk":355366.4476,"dot":420.11781,"eos":3589.45275,"eth":0.01228,"eur":356391.25289,"gbp":429275.06027,"hkd":311114.13213,"huf":483302.40632,"idr":343538.42758,"ils":284086.72882,"inr":385515.90264,"jpy":370460.77349,"krw":387745.09678,"kwd":315529.36994,"lkr":783642.79322,"ltc":17708.92732,"mmk":596681.32407,"mxn":374218.4574,"myr":333118.24429,"ngn":666966.22945,"nok":405242.11284,"nzd":363104.44575,"php":388006.4432,"pkr":682432.93371,"pln":406632.69119,"rub":288701.00391,"sar":307708.92693,"sek":413771.36058,"sgd":316957.32927,"thb":329084.66661,"try":1959879.65442,"twd":300649.74578,"uah":512118.41436,"usd":307052.47035,"vef":4765.74341,"vnd":326920.30986,"xag":259810.62684,"xau":217425.1203,"xdr":342708.07101,"xlm":1751.0144,"xrp":967.43494,"yfi":1770.03507,"zar":416179.38274,"bits":792.1343,"link":694.79729,"sats":792.1343},"atl_date":{"aed":"2015-10-20T00:00:00.0
2022-09-29 23:17:14.204 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  00Z","ars":"2015-10-20T00:00:00.000Z","aud":"2015-10-20T00:00:00.000Z","bch":"2017-11-12T00:00:00.000Z","bdt":"2015-10-20T00:00:00.000Z","bhd":"2015-10-20T00:00:00.000Z","bmd":"2015-10-20T00:00:00.000Z","bnb":"2021-04-12T21:29:59.636Z","brl":"2015-10-20T00:00:00.000Z","btc":"2015-10-20T00:00:00.000Z","cad":"2015-10-20T00:00:00.000Z","chf":"2015-10-20T00:00:00.000Z","clp":"2015-10-20T00:00:00.000Z","cny":"2015-10-20T00:00:00.000Z","czk":"2015-10-20T00:00:00.000Z","dkk":"2015-10-20T00:00:00.000Z","dot":"2021-03-03T00:43:22.672Z","eos":"2019-04-16T00:00:00.000Z","eth":"2019-10-22T00:00:00.000Z","eur":"2015-10-20T00:00:00.000Z","gbp":"2015-10-20T00:00:00.000Z","hkd":"2015-10-20T00:00:00.000Z","huf":"2015-10-20T00:00:00.000Z","idr":"2015-10-20T00:00:00.000Z","ils":"2015-10-20T00:00:00.000Z","inr":"2015-10-20T00:00:00.000Z","jpy":"2015-10-20T00:00:00.000Z","krw":"2015-10-20T00:00:00.000Z","kwd":"2015-10-20T00:00:00.000Z","lkr":"2015-10-20T00:00:00.000Z","ltc":"2015-10-20T00:00:00.000Z","mmk":"2015-10-20T00:00:00.000Z","mxn":"2015-10-20T00:00:00.000Z","myr":"2015-10-20T00:00:00.000Z","ngn":"2015-10-20T00:00:00.000Z","nok":"2015-10-20T00:00:00.000Z","nzd":"2015-10-20T00:00:00.000Z","php":"2015-10-20T00:00:00.000Z","pkr":"2015-10-20T00:00:00.000Z","pln":"2015-10-20T00:00:00.000Z","rub":"2015-10-20T00:00:00.000Z","sar":"2015-10-20T00:00:00.000Z","sek":"2015-10-20T00:00:00.000Z","sgd":"2015-10-20T00:00:00.000Z","thb":"2015-10-20T00:00:00.000Z","try":"2015-10-20T00:00:00.000Z","twd":"2015-10-20T00:00:00.000Z","uah":"2015-10-20T00:00:00.000Z","usd":"2015-10-20T00:00:00.000Z","vef":"2015-10-20T00:00:00.000Z","vnd":"2015-10-20T00:00:00.000Z","xag":"2015-10-20T00:00:00.000Z","xau":"2015-10-20T00:00:00.000Z","xdr":"2015-10-20T00:00:00.000Z","xlm":"2018-11-20T00:00:00.000Z","xrp":"2018-12-20T00:00:00.000Z","yfi":"2020-09-14T08:23:43.987Z","zar":"2015-10-20T00:00:00.000Z","bits":"2016-12-27T00:00:00.000Z","link":"2020-08-16T08:12:39.777Z","sats":"2016-12-27T00:00:00.000Z"},"market_cap":{"aed":588678312735,"ars":23575211784337,"aud":246789260237,"bch":1379649612,"bdt":16227787246549,"bhd":60484392955,"bmd":160269834085,"bnb":568709409,"brl":862956894645,"btc":8304975,"cad":219432161178,"chf":156945356916,"clp":155437698586916,"cny":1141922567853,"czk":4023894564091,"dkk":1218932223130,"dot":25173742292,"eos":133894394240,"eth":120734247,"eur":163880392907,"gbp":144890020266,"hkd":1258110184072,"huf":68922296809574,"idr":2433449012331294,"ils":571218517010,"inr":13072480016644,"jpy":23143865559628,"krw":228839504564108,"kwd":49691341518,"lkr":57635272268011,"ltc":3004117328,"mmk":336652713753904,"mxn":3226300515881,"myr":742930815899,"ngn":69242979117896,"nok":1716570859312,"nzd":280694984717,"php":9376585681499,"pkr":37174588015915,"pln":795311324963,"rub":9355734576084,"sar":602350451471,"sek":1791957782519,"sgd":229867971155,"thb":6073263810642,"try":2963693744908,"twd":5078390257990,"uah":5891659657593,"usd":160269834085,"vef":16047818487,"vnd":3807789742661065,"xag":8531799540,"xau":96469619,"xdr":126536559946,"xlm":1482763172064,"xrp":368688066064,"yfi":19809044,"zar":2879503039445,"bits":8304975286652,"link":20504464031,"sats":830497528665188},"market_cap_rank":2,"fully_diluted_valuation":{},"total_volume":{"aed":46417249173,"ars":1858723644318,"aud":19437771627,"bch":108845852,"bdt":1279560037881,"bhd":4769190708,"bmd":12637266675,"bnb":44838888,"brl":68093384022,"btc":655219,"cad":17290902216,"chf":12368257179,"clp":12246143270965,"cny":90040525056,"czk":316835193519,"dkk":95993713916,"dot":1985350519,"eos":10555801665,"eth":9529483,"eur":12908285496,"gbp":11391168994,"hkd":99201911532,"huf":5424767543645,"idr":191991153638799,"ils":45040545341,"inr":1030054659457,"jpy":1823579345290,"krw":18031831188747,"kwd":3918159258,"lkr":4544537714604,"ltc":237160338,"mmk":26545046013909,"mxn":254300349155,"myr":58580049670,"ngn":5459804694069,"nok":135242031263,"nzd":22106459055,"php":739874051994,"pkr":2931214005160,"pln":62661254942,"rub":7376991
2022-09-29 23:17:14.204 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  02576,"sar":47492528919,"sek":141133714545,"sgd":18120576685,"thb":479020420734,"try":233652003204,"twd":400456037653,"uah":464557005839,"usd":12637266675,"vef":1265369512,"vnd":300243989727995,"xag":670323970,"xau":7596893,"xdr":9977400059,"xlm":116925205829,"xrp":29089515792,"yfi":1558212,"zar":227086210205,"bits":655218716934,"link":1619080093,"sats":65521871693437},"high_24h":{"aed":4973.63,"ars":198868,"aud":2079.05,"bch":11.773638,"bdt":139087,"bhd":510.86,"bmd":1354.09,"bnb":4.792134,"brl":7279.84,"btc":0.06907022,"cad":1842.82,"chf":1320.82,"clp":1299605,"cny":9750.09,"czk":34349,"dkk":10366.43,"dot":210.325,"eos":1169,"eth":1.0,"eur":1393.81,"gbp":1249.69,"hkd":10628.26,"huf":580411,"idr":20548588,"ils":4780.86,"inr":110198,"jpy":195143,"krw":1933290,"kwd":420.06,"lkr":493782,"ltc":25.245497,"mmk":2848763,"mxn":27293,"myr":6258.42,"ngn":585074,"nok":14474.4,"nzd":2367.99,"php":79524,"pkr":316905,"pln":6709.72,"rub":79146,"sar":5100.48,"sek":15319.23,"sgd":1941.82,"thb":51237,"try":25063,"twd":42950,"uah":49857,"usd":1354.09,"vef":135.58,"vnd":32144471,"xag":71.82,"xau":0.815864,"xdr":1054.28,"xlm":12375,"xrp":3563,"yfi":0.16431094,"zar":24207,"bits":69070,"link":174.399,"sats":6907022},"low_24h":{"aed":4756.89,"ars":190525,"aud":2001.24,"bch":11.270123,"bdt":131131,"bhd":488.57,"bmd":1295.08,"bnb":4.641298,"brl":7006.76,"btc":0.06765578,"cad":1778.3,"chf":1270.07,"clp":1249671,"cny":9236.0,"czk":32695,"dkk":9889.95,"dot":205.119,"eos":1094,"eth":1.0,"eur":1329.98,"gbp":1178.27,"hkd":10166.37,"huf":558679,"idr":19680351,"ils":4604.82,"inr":105840,"jpy":187224,"krw":1854710,"kwd":401.75,"lkr":465729,"ltc":24.611387,"mmk":2720365,"mxn":26217,"myr":6003.35,"ngn":559527,"nok":13957.63,"nzd":2280.63,"php":75832,"pkr":296591,"pln":6462.57,"rub":75699,"sar":4867.34,"sek":14569.39,"sgd":1861.3,"thb":49310,"try":24003,"twd":41080,"uah":47608,"usd":1295.08,"vef":129.68,"vnd":30769334,"xag":69.38,"xau":0.783361,"xdr":989.33,"xlm":12068,"xrp":2854,"yfi":0.16060881,"zar":23366,"bits":67656,"link":164.068,"sats":6765578},"price_change_24h":-11.83334803686,"price_change_percentage_24h":-0.88414,"price_change_percentage_7d":5.8903,"price_change_percentage_14d":-18.88268,"price_change_percentage_30d":-14.69997,"price_change_percentage_60d":-21.81881,"price_change_percentage_200d":-48.57188,"price_change_percentage_1y":-52.60542,"market_cap_change_24h":-1883074944.348,"market_cap_change_percentage_24h":-1.1613,"price_change_24h_in_currency":{"aed":-43.477803841336936,"ars":-1452.7839606594061,"aud":-14.598266301851709,"bch":-0.3191656074238942,"bdt":-3157.0139196971722,"bhd":-4.278423030910233,"bmd":-11.833348036858979,"bnb":-0.062036729769069154,"brl":-21.468185148735756,"btc":0.00050147,"cad":-10.550924807533647,"chf":-7.746935187440158,"clp":1180.78,"cny":-185.36181450669392,"czk":-763.2339891952579,"dkk":-190.9483373677158,"dot":0.90134339,"eos":-51.45455859366416,"eth":0.0,"eur":-25.629058524687025,"gbp":-38.60026984729461,"hkd":-92.72121360557139,"huf":-1025.5597266132245,"idr":-199184.8722877875,"ils":2.54,"inr":-827.9574713049369,"jpy":-1528.375593691453,"krw":-14148.954239486251,"kwd":-4.00618272465573,"lkr":-11010.334757892007,"ltc":-0.003651073281083939,"mmk":-29260.831142453942,"mxn":-460.7003340759329,"myr":-22.062681662899195,"ngn":-4162.215594212059,"nok":-138.02840611977444,"nzd":-24.43831011478096,"php":-1059.5340164771624,"pkr":-9169.065696637379,"pln":-64.32445151813408,"rub":-791.2949942103587,"sar":-55.80486805068995,"sek":-202.9515186032604,"sgd":-20.609624501224516,"thb":-494.21111327944527,"try":-254.08855442590357,"twd":-410.44598771923484,"uah":-513.9471064935788,"usd":-11.833348036858979,"vef":-1.184873138930783,"vnd":-254720.7004112266,"xag":-0.9398096623575327,"xau":-0.009429049395634716,"xdr":43.89,"xlm":124.277,"xrp":12.002075,"yfi":0.00175685,"zar":-43.57745128017268,"bits":501.47,"link":5.890609,"sats":50147},"price_change_percentage_1h_in_currency":{"aed":0.15089,"ars":0.13778,"aud":-0.0759,"bch":0.1328
2022-09-29 23:17:14.204 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  1,"bdt":0.15089,"bhd":0.15381,"bmd":0.15089,"bnb":0.31886,"brl":0.02651,"btc":-0.19168,"cad":0.04939,"chf":0.10046,"clp":0.64943,"cny":0.14245,"czk":-0.16392,"dkk":-0.00409,"dot":-0.12635,"eos":0.3329,"eth":0.0,"eur":-0.00672,"gbp":-0.31459,"hkd":0.15242,"huf":-0.10193,"idr":0.32487,"ils":0.15089,"inr":-0.0536,"jpy":-0.00722,"krw":-0.1415,"kwd":0.16155,"lkr":0.15089,"ltc":0.1908,"mmk":0.15089,"mxn":-0.13175,"myr":0.15089,"ngn":0.15089,"nok":-0.01224,"nzd":-0.13558,"php":0.16542,"pkr":0.15089,"pln":-0.09693,"rub":0.09498,"sar":0.14439,"sek":-0.06877,"sgd":0.07314,"thb":-0.07745,"try":-0.02998,"twd":0.09719,"uah":0.15089,"usd":0.15089,"vef":0.15089,"vnd":0.15089,"xag":-0.33241,"xau":-0.1961,"xdr":0.15089,"xlm":0.14991,"xrp":0.1204,"yfi":-0.3891,"zar":-0.0947,"bits":-0.19168,"link":0.64488,"sats":-0.19168},"price_change_percentage_24h_in_currency":{"aed":-0.88441,"ars":-0.73908,"aud":-0.71037,"bch":-2.71746,"bdt":-2.29641,"bhd":-0.84736,"bmd":-0.88414,"bnb":-1.30086,"brl":-0.29944,"btc":0.73445,"cad":-0.57794,"chf":-0.59315,"clp":0.09194,"cny":-1.92341,"czk":-2.24334,"dkk":-1.85971,"dot":0.43437,"eos":-4.43756,"eth":0.0,"eur":-1.85631,"gbp":-3.12715,"hkd":-0.88254,"huf":-0.17977,"idr":-0.97865,"ils":0.05376,"inr":-0.7599,"jpy":-0.79209,"krw":-0.74195,"kwd":-0.96463,"lkr":-2.25593,"ltc":-0.01466,"mmk":-1.03918,"mxn":-1.69654,"myr":-0.3575,"ngn":-0.72099,"nok":-0.96289,"nzd":-1.04214,"php":-1.34585,"pkr":-2.89367,"pln":-0.96844,"rub":-1.0115,"sar":-1.10697,"sek":-1.35138,"sgd":-1.07187,"thb":-0.97327,"try":-1.02533,"twd":-0.96695,"uah":-1.04292,"usd":-0.88414,"vef":-0.88414,"vnd":-0.80171,"xag":-1.31801,"xau":-1.16856,"xdr":4.37436,"xlm":1.02288,"xrp":0.3946,"yfi":1.08573,"zar":-0.18247,"bits":0.73445,"link":3.59034,"sats":0.73445},"price_change_percentage_7d_in_currency":{"aed":5.89117,"ars":7.4885,"aud":7.67206,"bch":-0.43697,"bdt":3.46304,"bhd":5.99253,"bmd":5.8903,"bnb":-0.5958,"brl":10.32091,"btc":1.76262,"cad":7.4933,"chf":7.13271,"clp":9.41023,"cny":7.03198,"czk":5.87044,"dkk":6.32678,"dot":2.34496,"eos":5.28387,"eth":0.0,"eur":6.34653,"gbp":7.42929,"hkd":5.89273,"huf":9.75554,"idr":6.82874,"ils":9.03884,"inr":7.81945,"jpy":5.88937,"krw":8.09616,"kwd":6.14982,"lkr":5.99713,"ltc":1.7576,"mmk":5.83582,"mxn":6.35001,"myr":7.82087,"ngn":6.42484,"nok":9.29245,"nzd":8.0103,"php":6.83831,"pkr":2.37046,"pln":7.98091,"rub":0.34636,"sar":5.78002,"sek":6.57453,"sgd":7.02044,"thb":7.81124,"try":6.74316,"twd":6.67636,"uah":5.32161,"usd":5.8903,"vef":5.8903,"vnd":6.16706,"xag":9.30238,"xau":6.07558,"xdr":12.62584,"xlm":8.847,"xrp":-2.86754,"yfi":5.04499,"zar":7.2203,"bits":1.76262,"link":-8.96495,"sats":1.76262},"price_change_percentage_14d_in_currency":{"aed":-18.88187,"ars":-16.48117,"aud":-15.74679,"bch":-15.978,"bdt":-15.23543,"bhd":-18.79188,"bmd":-18.88268,"bnb":-19.53223,"brl":-15.36425,"btc":-14.82472,"cad":-15.66593,"chf":-17.47531,"clp":-14.59967,"cny":-16.9835,"czk":-17.08519,"dkk":-17.29433,"dot":-8.14936,"eos":0.04398,"eth":0.0,"eur":-17.29167,"gbp":-15.59388,"hkd":-18.87132,"huf":-14.32045,"idr":-17.41521,"ils":-15.60021,"inr":-16.78442,"jpy":-18.08262,"krw":-16.82335,"kwd":-18.55169,"lkr":-20.08369,"ltc":-8.56072,"mmk":-18.86633,"mxn":-18.24812,"myr":-16.93852,"ngn":-18.00487,"nok":-13.97995,"nzd":-14.63813,"php":-16.81976,"pkr":-19.88676,"pln":-14.89079,"rub":-20.71638,"sar":-18.92107,"sek":-15.25789,"sgd":-17.24537,"thb":-16.10415,"try":-17.82667,"twd":-17.29373,"uah":-19.26121,"usd":-18.88268,"vef":-18.88268,"vnd":-18.34306,"xag":-15.4673,"xau":-17.20657,"xdr":-10.02264,"xlm":-20.98016,"xrp":-36.06806,"yfi":-7.84499,"zar":-16.64634,"bits":-14.82472,"link":-22.57874,"sats":-14.82472},"price_change_percentage_30d_in_currency":{"aed":-14.69886,"ars":-9.29081,"aud":-9.38001,"bch":-12.08887,"bdt":-9.15403,"bhd":-14.61966,"bmd":-14.69997,"bnb":-13.31736,"brl":-8.60019,"btc":-10.15348,"cad":-10.24677,"chf":-13.69782,"clp":-6.33535,"cny":-12.02046,"czk":-12.96282,"dkk":-12.80149,"dot":-2.33077,"eos":6.55753,"eth":0.0,"eur":-12.7
2022-09-29 23:17:14.204 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  8778,"gbp":-9.88107,"hkd":-14.67068,"huf":-10.11393,"idr":-12.95091,"ils":-8.45443,"inr":-13.05607,"jpy":-11.18658,"krw":-9.57923,"kwd":-14.17068,"lkr":-14.59431,"ltc":-10.61162,"mmk":-14.72084,"mxn":-14.16701,"myr":-11.83762,"ngn":-13.54158,"nok":-6.25046,"nzd":-8.09767,"php":-11.2167,"pkr":-10.96045,"pln":-10.58487,"rub":-19.36226,"sar":-14.64571,"sek":-10.60521,"sgd":-12.34492,"thb":-11.02559,"try":-13.27485,"twd":-11.03479,"uah":-15.13634,"usd":-14.69997,"vef":-14.69997,"vnd":-13.52452,"xag":-14.96479,"xau":-10.79741,"xdr":-8.84534,"xlm":-16.30885,"xrp":-34.37399,"yfi":-4.41311,"zar":-9.02614,"bits":-10.15348,"link":-26.57409,"sats":-10.15348},"price_change_percentage_60d_in_currency":{"aed":-21.81998,"ars":-12.35827,"aud":-15.91599,"bch":-2.60469,"bdt":-16.51183,"bhd":-21.70076,"bmd":-21.81881,"bnb":-20.28887,"brl":-18.56176,"btc":-4.15615,"cad":-16.39774,"chf":-19.64033,"clp":-16.0519,"cny":-17.40444,"czk":-18.5794,"dkk":-18.41482,"dot":0.91078,"eos":-14.62723,"eth":0.0,"eur":-18.33505,"gbp":-14.20015,"hkd":-21.81901,"huf":-15.12995,"idr":-19.82906,"ils":-18.26299,"inr":-19.53414,"jpy":-15.30589,"krw":-14.40444,"kwd":-21.01301,"lkr":-21.58136,"ltc":-11.41509,"mmk":-11.43538,"mxn":-22.75524,"myr":-18.56895,"ngn":-18.77894,"nok":-13.37836,"nzd":-14.11993,"php":-17.3113,"pkr":-24.15905,"pln":-16.34399,"rub":-26.68565,"sar":-21.77689,"sek":-13.93475,"sgd":-18.79816,"thb":-18.2803,"try":-19.31242,"twd":-17.45152,"uah":-22.25601,"usd":-21.81881,"vef":-21.81881,"vnd":-20.41658,"xag":-15.60877,"xau":-16.9929,"xdr":-16.14998,"xlm":-13.64825,"xrp":-30.3295,"yfi":13.03559,"zar":-15.40529,"bits":-4.15615,"link":-21.74235,"sats":-4.15615},"price_change_percentage_200d_in_currency":{"aed":-48.57265,"ars":-30.47244,"aud":-42.3257,"bch":28.94234,"bdt":-39.38461,"bhd":-48.49122,"bmd":-48.57188,"bnb":-31.85766,"brl":-45.39702,"btc":3.89145,"cad":-44.78995,"chf":-46.14586,"clp":-38.16515,"cny":-42.20238,"czk":-43.97995,"dkk":-42.69987,"dot":46.79011,"eos":-15.29351,"eth":0.0,"eur":-42.66759,"gbp":-39.55733,"hkd":-48.43756,"huf":-36.98034,"idr":-45.46524,"ils":-43.61622,"inr":-45.38935,"jpy":-36.72699,"krw":-40.68278,"kwd":-47.48637,"lkr":-27.34246,"ltc":2.15097,"mmk":-39.13792,"mxn":-50.53518,"myr":-43.16485,"ngn":-46.56838,"nok":-38.89942,"nzd":-38.7661,"php":-42.52408,"pkr":-33.24705,"pln":-41.86557,"rub":-77.59619,"sar":-48.48128,"sek":-41.07731,"sgd":-45.84701,"thb":-41.4981,"try":-35.59816,"twd":-42.64215,"uah":-35.5807,"usd":-48.57188,"vef":-48.57188,"vnd":-46.59246,"xag":-29.5651,"xau":-38.5392,"xdr":-43.28904,"xlm":-11.69434,"xrp":-6.98716,"yfi":16.73474,"zar":-38.53348,"bits":3.89145,"link":-13.10095,"sats":3.89145},"price_change_percentage_1y_in_currency":{"aed":-52.60742,"ars":-29.28061,"aud":-47.20669,"bch":93.70257,"bdt":-43.97175,"bhd":-52.55753,"bmd":-52.60542,"bnb":-44.14495,"brl":-52.96593,"btc":1.3196,"cad":-48.8618,"chf":-50.08062,"clp":-42.71186,"cny":-47.72657,"czk":-45.54419,"dkk":-43.42289,"dot":95.9204,"eos":46.20441,"eth":0.0,"eur":-43.42738,"gbp":-42.15886,"hkd":-52.19988,"huf":-33.90685,"idr":-49.71313,"ils":-47.37833,"inr":-47.93707,"jpy":-38.68681,"krw":-43.06689,"kwd":-51.2237,"lkr":-14.67498,"ltc":25.14698,"mmk":-46.80559,"mxn":-53.10786,"myr":-47.47219,"ngn":-50.15515,"nok":-41.43798,"nzd":-42.31604,"php":-45.74827,"pkr":-35.40287,"pln":-40.76225,"rub":-62.03667,"sar":-52.5165,"sek":-39.39164,"sgd":-49.92534,"thb":-46.90977,"try":-1.25908,"twd":-45.88485,"uah":-34.49027,"usd":-52.60542,"vef":-52.60542,"vnd":-50.5109,"xag":-43.4973,"xau":-50.58407,"xdr":-47.01463,"xlm":12.88397,"xrp":-1.9942,"yfi":64.92974,"zar":-43.63153,"bits":1.3196,"link":34.20475,"sats":1.3196},"market_cap_change_24h_in_currency":{"aed":-6918240538.055542,"ars":-233103774284.54688,"aud":-2038918948.880371,"bch":-41826894.277243376,"bdt":-427990250037.25195,"bhd":-687954129.0587006,"bmd":-1883074944.3485718,"bnb":-9290299.869598627,"brl":-5112488550.455811,"btc":32229,"cad":-1693490723.7938843,"chf":-1156485986.4953308,"clp":-133423864514
2022-09-29 23:17:14.204 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  .71875,"cny":-25659453610.16577,"czk":-96158382254.00098,"dkk":-24021389201.534912,"dot":117658157,"eos":-6523338939.160385,"eth":14937,"eur":-3266339261.455475,"gbp":-4420702673.499939,"hkd":-14765828868.483643,"huf":-137978534204.25,"idr":-31758044868821.0,"ils":-1293948133.871704,"inr":-154179700498.3086,"jpy":-223233446412.83203,"krw":-2488678813820.9688,"kwd":-610761003.2442169,"lkr":-1495564826794.6875,"ltc":-6886112.174203873,"mmk":-4489086057831.3125,"mxn":-64191930869.57373,"myr":-4756247633.319702,"ngn":-698435133545.8906,"nok":-19126364707.588867,"nzd":-3293944804.0578613,"php":-165059345250.20312,"pkr":-1215113196680.4375,"pln":-8734414395.597778,"rub":-122103280961.75586,"sar":-8398642538.919312,"sek":-26638332559.09326,"sgd":-3065006971.0610046,"thb":-78804423529.52148,"try":-38632389078.48584,"twd":-69526628670.31934,"uah":-78787751444.24219,"usd":-1883074944.3485718,"vef":-188552294.17759514,"vnd":-41537962794006.5,"xag":-88792316.65958023,"xau":-1219401.5122876912,"xdr":4963227166,"xlm":12676465667,"xrp":758425865,"yfi":231218,"zar":-13122273455.515625,"bits":32228768819,"link":747262057,"sats":3222876881867},"market_cap_change_percentage_24h_in_currency":{"aed":-1.16156,"ars":-0.97909,"aud":-0.81941,"bch":-2.9425,"bdt":-2.56962,"bhd":-1.12462,"bmd":-1.1613,"bnb":-1.60732,"brl":-0.58895,"btc":0.38958,"cad":-0.76585,"chf":-0.73148,"clp":-0.08576,"cny":-2.19766,"czk":-2.33391,"dkk":-1.93261,"dot":0.46958,"eos":-4.64567,"eth":0.01237,"eur":-1.95417,"gbp":-2.96074,"hkd":-1.16004,"huf":-0.19979,"idr":-1.28825,"ils":-0.22601,"inr":-1.16567,"jpy":-0.95533,"krw":-1.07582,"kwd":-1.21419,"lkr":-2.52925,"ltc":-0.2287,"mmk":-1.3159,"mxn":-1.95083,"myr":-0.63613,"ngn":-0.9986,"nok":-1.10194,"nzd":-1.15988,"php":-1.72988,"pkr":-3.16521,"pln":-1.08631,"rub":-1.2883,"sar":-1.37514,"sek":-1.46477,"sgd":-1.31583,"thb":-1.28094,"try":-1.28675,"twd":-1.35058,"uah":-1.31963,"usd":-1.1613,"vef":-1.1613,"vnd":-1.0791,"xag":-1.03,"xau":-1.24825,"xdr":4.0825,"xlm":0.86229,"xrp":0.20613,"yfi":1.18102,"zar":-0.45365,"bits":0.38958,"link":3.78223,"sats":0.38958},"total_supply":120732090.871192,"max_supply":null,"circulating_supply":120734246.871192,"last_updated":"2022-09-29T16:15:38.516Z"},"public_interest_stats":{"alexa_rank":8793,"bing_matches":null},"status_updates":[],"last_updated":"2022-09-29T16:15:38.516Z"}
2022-09-29 23:17:14.204 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- END HTTP (30345-byte body)
2022-09-29 23:17:14.466 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- 429 https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false (739ms)
2022-09-29 23:17:14.466 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  date: Thu, 29 Sep 2022 16:17:14 GMT
2022-09-29 23:17:14.467 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  content-type: application/json
2022-09-29 23:17:14.467 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  retry-after: 120
2022-09-29 23:17:14.467 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cache-control: no-cache
2022-09-29 23:17:14.467 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  vary: Accept-Encoding
2022-09-29 23:17:14.467 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  server: cloudflare
2022-09-29 23:17:14.467 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-ray: 7526071f487ce68e-HKG
2022-09-29 23:17:14.469 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  {"status":{"error_code":429,"error_message":"You've exceeded the Rate Limit. Please visit https://www.coingecko.com/en/api/pricing to subscribe to our API plans for higher rate limits."}}
2022-09-29 23:17:14.469 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- END HTTP (187-byte body)
2022-09-29 23:17:14.501 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:14.501 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:14.523 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:14.524 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:14.776 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- 429 https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false (275ms)
2022-09-29 23:17:14.777 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  date: Thu, 29 Sep 2022 16:17:14 GMT
2022-09-29 23:17:14.777 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  content-type: application/json
2022-09-29 23:17:14.777 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  retry-after: 120
2022-09-29 23:17:14.777 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cache-control: no-cache
2022-09-29 23:17:14.778 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  vary: Accept-Encoding
2022-09-29 23:17:14.778 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  server: cloudflare
2022-09-29 23:17:14.778 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-ray: 752607218bede68e-HKG
2022-09-29 23:17:14.780 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  {"status":{"error_code":429,"error_message":"You've exceeded the Rate Limit. Please visit https://www.coingecko.com/en/api/pricing to subscribe to our API plans for higher rate limits."}}
2022-09-29 23:17:14.780 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- END HTTP (187-byte body)
2022-09-29 23:17:15.199 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- 429 https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false (675ms)
2022-09-29 23:17:15.200 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  date: Thu, 29 Sep 2022 16:17:14 GMT
2022-09-29 23:17:15.200 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  content-type: application/json
2022-09-29 23:17:15.200 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  retry-after: 119
2022-09-29 23:17:15.200 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cache-control: no-cache
2022-09-29 23:17:15.200 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  vary: Accept-Encoding
2022-09-29 23:17:15.201 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  server: cloudflare
2022-09-29 23:17:15.201 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-ray: 752607231ebbe68e-HKG
2022-09-29 23:17:15.202 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  {"status":{"error_code":429,"error_message":"You've exceeded the Rate Limit. Please visit https://www.coingecko.com/en/api/pricing to subscribe to our API plans for higher rate limits."}}
2022-09-29 23:17:15.203 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- END HTTP (187-byte body)
2022-09-29 23:17:15.218 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:15.218 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:15.246 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:15.246 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:15.692 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- 429 https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false (473ms)
2022-09-29 23:17:15.692 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  date: Thu, 29 Sep 2022 16:17:15 GMT
2022-09-29 23:17:15.692 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  content-type: application/json
2022-09-29 23:17:15.693 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  retry-after: 119
2022-09-29 23:17:15.693 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cache-control: no-cache
2022-09-29 23:17:15.693 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  vary: Accept-Encoding
2022-09-29 23:17:15.693 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  server: cloudflare
2022-09-29 23:17:15.693 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-ray: 75260725eb8de68e-HKG
2022-09-29 23:17:15.694 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  {"status":{"error_code":429,"error_message":"You've exceeded the Rate Limit. Please visit https://www.coingecko.com/en/api/pricing to subscribe to our API plans for higher rate limits."}}
2022-09-29 23:17:15.694 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- END HTTP (187-byte body)
2022-09-29 23:17:16.084 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- 429 https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false (837ms)
2022-09-29 23:17:16.084 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  date: Thu, 29 Sep 2022 16:17:15 GMT
2022-09-29 23:17:16.084 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  content-type: application/json
2022-09-29 23:17:16.085 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  retry-after: 119
2022-09-29 23:17:16.085 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cache-control: no-cache
2022-09-29 23:17:16.085 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  vary: Accept-Encoding
2022-09-29 23:17:16.085 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  server: cloudflare
2022-09-29 23:17:16.085 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  cf-ray: 75260728d904e68e-HKG
2022-09-29 23:17:16.086 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  {"status":{"error_code":429,"error_message":"You've exceeded the Rate Limit. Please visit https://www.coingecko.com/en/api/pricing to subscribe to our API plans for higher rate limits."}}
2022-09-29 23:17:16.086 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- END HTTP (187-byte body)
2022-09-29 23:17:16.116 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:16.117 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:16.140 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> GET https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false
2022-09-29 23:17:16.140 15724-15928 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  --> END GET
2022-09-29 23:17:16.513 15724-15926 okhttp.OkHttpClient     co.nimblehq.compose.crypto           I  <-- 429 https://api.coingecko.com/api/v3/coins/ethereum?localization=false&tickers=false&market_data=true&community_data=false&developer_data=false&sparkline=false (395ms)
```

</details>

Reply:
Fixed in https://github.com/nimblehq/jetpack-compose-crypto/pull/45/commits/b2190d1c7a2fb2035394961d16f0f8e3e182af9c